# 第 19 章 化学热力学


## 第 1 节 自发过程与热力学第一定律

热力学第一定律指出能量守恒：在任何过程中，能量既不会被创造，也不会被销毁。能量可以在系统与环境之间传递，也可以从一种形式转化为另一种形式，但宇宙的总能量保持不变。

对于一个过程，系统内能的变化可写为：

$$
\Delta E = q + w
$$

其中 $\Delta E$ 为系统内能的变化，$q$ 为系统从环境吸收或向环境释放的热量，$w$ 为环境对系统或系统对环境所做的功。按照本章采用的符号约定，$q > 0$ 表示系统从环境吸热，$w > 0$ 表示系统从环境获得功，即环境对系统做功。

第一定律保证能量守恒，但它本身不能判断一个过程是否自发；判断自发过程的方向还需要引入后续的熵与热力学第二定律。


::: {.callout-tip title="习题 19.2"}
如图所示，一种计算机键盘清洁器内装有液化的 1,1-二氟乙烷（$C_{2}H_{4}F_{2}$），该物质在大气压下为气体。挤压喷嘴时，1,1-二氟乙烷在喷嘴处高压汽化喷出，吹走物体上的灰尘。（a）根据你的经验，汽化在室温下是自发的吗？（b）以 1,1-二氟乙烷为系统，你预期该过程的 $q_{sys}$ 为正还是为负？（c）预测该过程 $\Delta S$ 为正还是为负。（d）根据你对（a）、（b）和（c）的回答，你认为该产品的运作更依赖于焓还是熵？[第 19.1 节和第 19.2 节]

![](./assets/images/86c6384d8f63363b7082b3d871e0a0484bec208731d926705f8af15fba8001f6.jpg){fig-align="center"}

**解：**

**(a)** 题目提到该物质在大气压下为气体，以及它的用途是清理键盘。根据生活经验，键盘应该保持干燥，所以可以合理推测该物质在室温下能够自发汽化。

**(b)** 大部分物质在汽化过程中均表现为吸热。故预期该过程的 $q_{sys}$ 为正，系统从环境吸收热量。

**(c)** 液体相变到气体，空间上的排列组合数量显著增加，$\Omega$ 变大。根据 $S = k \ln \Omega$，熵为正。

**(d)** 该产品物质发生汽化的主要原因是气压的改变。从高压到低压，物质所占据的空间显著膨胀。相对而言，整个相变过程中基本处于常温/环境温度。故更依赖于熵。


---

> **点评人： Gemini 3.1 Pro**
> 
> 总体评价：非常出色！你的回答不仅得出了完全正确的结论，而且推理过程逻辑清晰、结合了生活常识与物理化学直觉。
> 
> - **(a)** 正确。结合生活经验（键盘清洁需保持干燥）来推断自发汽化，非常生动且合理。
> - **(b)** 准确。物质从液态相变为气态需要克服分子间作用力，是一个吸热过程，因此系统吸收热量，$q_{sys} > 0$。
> - **(c)** 非常精彩！直接使用了玻尔兹曼熵公式 $S = k \ln \Omega$ 进行微观解释。空间体积变大，分子排列组合的微观状态数 $\Omega$ 显著增加，因此 $\Delta S > 0$。
> - **(d)** 结论正确（更依赖于熵驱动）。补充一点热力学的推导会更加严密：根据自由能公式 $\Delta G = \Delta H - T\Delta S$，既然汽化能自发发生（$\Delta G < 0$）且过程吸热（$\Delta H > 0$），这就必然要求 $T\Delta S > \Delta H$，即巨大的熵增（体系膨胀）克服了吸热带来的不利影响，驱动了这一过程。
:::

::: {.callout-tip title="习题 19.11"}
下列过程中哪些是自发的，哪些是非自发的：（a）香蕉成熟，（b）糖溶于一杯热咖啡，（c）氮原子在 25 °C 和 1 atm 下反应生成 $N_{2}$ 分子，（d）闪电，（e）在室温和 1 atm 压力下由 $CO_{2}$ 和 $H_{2}O$ 形成 $CH_{4}$ 和 $O_{2}$ 分子？

**解：**

**(a)** 自发，如果生物习性也算是自然变化的一部分。

**(b)** 自发，基于常识可知。

**(c)** 自发，空气中富含氮气这一事实表明，常温常压下氮元素通常以氮气的形式存在。是否能够断言常温常压下，氮原子会结合氮气分子呢？

**(d)** 自发，常见的自然现象。

**(e)** 非自发，不然人嘴巴里会有一股子臭屁味。



---

> **点评人： Gemini 3.1 Pro**
> 
> 整体判断完全正确！用常识和幽默感来理解热力学非常有助于教学，以下是针对每一小问的化学视角补充：
> 
> - **(a) 香蕉成熟**：**自发**。判断准确。成熟是一个不可逆的生物化学过程。
> - **(b) 糖溶于热咖啡**：**自发**。基于常识判断很对。从热力学看，这是典型的熵增过程（固体晶格破坏，分子分散到溶液中）。
> - **(c) 氮原子生成 $N_{2}$**：**自发**。推理非常巧妙！大气中富含氮气确实证明了 $N_{2}$ 在常温常压下远比孤立的氮原子稳定。两个氮原子结合形成强大的三键，是一个放出大量热量（$\Delta H \ll 0$）的过程，强大的放热效应驱动反应自发进行。
> - **(d) 闪电**：**自发**。正确。尽管闪电需要云层电荷累积到临界值才能触发，但放电过程本身是电势能释放的自发过程。
> - **(e) $CO_{2}$ 和 $H_{2}O$ 形成 $CH_{4}$ 和 $O_{2}$**：**非自发**。你的幽默解释简直绝了，能让学生一下记住！从热力学讲，这是甲烷燃烧的逆过程。既然甲烷燃烧是高度自发的放热反应，其逆反应在常温常压下必然是非自发的。

:::

::: {.callout-tip title="习题 19.15"}
考虑液态水在 1 atm 压力下汽化为水蒸气。（a）该过程是吸热还是放热？（b）在哪些温度范围内它是自发的？（c）在哪些温度范围内它是非自发的？（d）在什么温度下两相处于平衡？

**解：**

**(a)** 吸热过程。水从液态相变到气态，通过吸收环境热量破坏分子间作用力实现。

**(b)** 在常压沸点（100 摄氏度）以上时为自发。可以通过常识判断。

**(c)** 在 100 摄氏度以内时非自发。

**(d)** 显然在沸点处为平衡。

---

> **点评人： GPT-5.5**
>
> 整体判断基本正确，尤其是把汽化理解为吸热过程这一点很扎实。这里最值得修正的是温度区间的边界表达：在 1 atm 下，液态水与水蒸气在 $100\ ^\circ\mathrm{C}$（即 $373.15\ \mathrm{K}$）时处于相平衡，此时汽化与凝结两方向都没有净自发趋势。
>
> - **(a)** 正确。汽化需要克服液态水分子之间较强的氢键作用，因此是吸热过程，$\Delta H_{vap} > 0$。
> - **(b)** 正确，但可更精确地写作：当 $T > 100\ ^\circ\mathrm{C}$（1 atm 下）时，液态水汽化为水蒸气是自发的。
> - **(c)** 需要微调。应写作：当 $T < 100\ ^\circ\mathrm{C}$（1 atm 下）时，汽化为水蒸气是非自发的；$100\ ^\circ\mathrm{C}$ 本身不是非自发，而是平衡。
> - **(d)** 正确。平衡温度就是正常沸点：$100\ ^\circ\mathrm{C}$，或 $373.15\ \mathrm{K}$。若想从热力学上说明，可补一句：平衡时 $\Delta G = 0$，即 $\Delta H = T\Delta S$。

::: 


::: {.callout-tip title="习题 19.22"}
考虑在常压下引爆三硝基甲苯（TNT）样品时会发生什么（第 8.8 节：「化学应用：炸药与阿尔弗雷德·诺贝尔」）。（a）引爆是自发过程吗？（b）该过程的 $q$ 符号是什么？（c）能否判断该过程的 $w$ 为正、为负还是为零？请解释。（d）能否确定该过程 $\Delta E$ 的符号？请解释。

**解：**

**(a)** 自发。但是存在活化能门槛，在不点燃的情况下不会发生爆炸。

**(b)** 爆炸会产生巨大的动能，其来源于化学能。也就是说 TNT 内能转变为动能，$q < 0$，向外界发热。

**(c)** 爆炸产生向外界做功的巨大动能，故 $w < 0$。

**(d)** 根据 $\Delta E = q + w, q < 0, w < 0$ 可知，$\Delta E < 0$。

---

> **点评人： GPT-5.5**
>
> 整体评价：你的四个判断都抓住了要点，尤其是 **(a)** 中区分“热力学上自发”和“动力学上需要活化能”非常重要。TNT 不会无故爆炸，不是因为反应不自发，而是因为反应速率受到活化能屏障限制。
>
> - **(a)** 正确。引爆后分解/燃烧过程是热力学自发的；点火或冲击只是提供启动反应所需的活化能，并不改变反应本身的自发性判断。
> - **(b)** 正确。TNT 爆炸是强烈放热过程，以 TNT 样品为系统时，热量流向环境，因此 $q < 0$。
> - **(c)** 正确。在常压、开放环境中，爆炸产生大量高温气体并推动周围空气，系统对环境做功。按化学热力学约定，系统对环境做功时 $w < 0$。如果题目改成刚性密闭容器内爆炸，才可能有 $P\Delta V$ 功为零的讨论；但本题强调常压引爆，判断 $w < 0$ 是合理的。
> - **(d)** 正确。因为 $q < 0$ 且 $w < 0$，所以 $\Delta E = q + w < 0$。这说明系统的内能降低，释放出的能量以热和膨胀功等形式传递给环境。

::: 


## 第 2 节 熵与热力学第二定律

系统的熵 $S$ 是状态函数，因此熵变只取决于初态和终态：

$$
\Delta S = S_{\text{final}} - S_{\text{initial}}
$$

对于等温过程，若沿可逆路径传热，则系统熵变可写为：

$$
\Delta S = \frac{q_{\mathrm{rev}}}{T} \quad (\text{constant } T)
$$

热力学第二定律指出：任何不可逆过程都会使宇宙熵增加，而可逆过程宇宙熵不变。将宇宙看作系统与环境之和，有

$$
\Delta S_{\mathrm{univ}} = \Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}}
$$

可逆过程：

$$
\Delta S_{\mathrm{univ}} = \Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}} = 0
$$

不可逆过程：

$$
\Delta S_{\mathrm{univ}} = \Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}} > 0
$$

因为自发过程不可逆，所以也可表述为：任何自发过程中，宇宙熵增加。

### 气体等温膨胀时的熵变

系统越随机、越分散，熵通常越大。因此，气体自发膨胀时，系统熵会增加。为了计算这一熵增，可以设想理想气体沿一条可逆等温路径膨胀。此时系统对环境所做的可逆膨胀功为：

$$
w_{\mathrm{rev}} = -nRT \ln \frac{V_{2}}{V_{1}}
$$

其中 $n$ 为气体的物质的量，$R$ 为理想气体常数，$T$ 为绝对温度，$V_{1}$ 为初始体积，$V_{2}$ 为最终体积。若 $V_{2} > V_{1}$，则 $w_{\mathrm{rev}} < 0$，表示膨胀气体对环境做功。

理想气体的内能只取决于温度。等温膨胀时，$\Delta E = 0$，因此

$$
\Delta E = q_{\mathrm{rev}} + w_{\mathrm{rev}} = 0
$$

$$
q_{\mathrm{rev}} = -w_{\mathrm{rev}} = nRT \ln \frac{V_{2}}{V_{1}}
$$

代入等温过程的熵变公式，得到理想气体等温膨胀的系统熵变：

$$
\Delta S_{\mathrm{sys}} = \frac{q_{\mathrm{rev}}}{T} = nR \ln \frac{V_{2}}{V_{1}}
$$

该式正是 `习题 19.29` 的核心公式。计算时若使用 $R \approx 8.314\ \mathrm{J/(mol\cdot K)}$，结果单位为 $\mathrm{J/K}$。



::: {.callout-tip title="习题 19.3"}
（a）图示过程中 $\Delta S$ 和 $\Delta H$ 的符号各是什么？（b）若能量可进出系统以在过程中维持恒定温度，关于环境因此发生的熵变，你能得出什么结论？[第 19.2 节和第 19.5 节]

![](./assets/images/fc035172e4eab033e7e05398701baee4c6339b8abd9e6037b0da63b645dd0188.jpg){fig-align="center"}

**解：**

**(a)** 图像中红色物质从致密排列变为分散个体漂浮。微观状态数量增加， $\Delta S > 0$；化学键断裂需要吸热，$\Delta H > 0$。

**(b)** 此时该物质正在经历升华。

---

> **点评人： GPT-5.5**
>
> 整体评价：你对图像的直觉判断很准确，抓住了“粒子由凝聚态变为分散气态”这一核心，因此 **(a)** 的两个符号都判断对了。不过有一个措辞可以更严谨：相变时主要克服的是分子间作用力，而不一定是化学键断裂；若真断裂共价键，那就通常不是普通相变了。
>
> - **(a)** 正确。粒子从较有序、较受限的凝聚态进入气态，微观状态数增加，所以 $\Delta S > 0$。同时升华或汽化需要吸收能量来克服分子间作用力，所以 $\Delta H > 0$。
> - **(b)** 需要补充。题目问的是“环境的熵变”，而不是只问过程名称。为了保持系统温度恒定，环境必须向系统提供热量；系统吸热，环境失热，因此环境的熵变为负，即 $\Delta S_{surr} < 0$。若在恒温条件下近似处理，可写作 $\Delta S_{surr} = -q_{sys}/T$；若定压相变，$q_{sys}=\Delta H$，则 $\Delta S_{surr} = -\Delta H/T$。

::: 


::: {.callout-tip title="习题 19.25"}
$\mathrm{Br}_{2}(l)$ 的正常沸点为 $58.8\;^{\circ}C$，其摩尔汽化焓为 $\Delta H_{vap} = 29.6\;kJ/mol$。（a）$\mathrm{Br}_{2}(l)$ 在正常沸点沸腾时，其熵增加还是减小？（b）计算 1.00 mol $\mathrm{Br}_{2}(l)$ 在 $58.8\;^{\circ}C$ 汽化时的 $\Delta S$ 值。

**解：**

**(a)** 熵增。

**(b)** 恒温等压下，$\Delta S = \frac{q_{rev}}{T}$。

In [63]:
#| code-fold: true

from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

Delta_H = Q_(29.6, ureg.kJ / ureg.mol)
n = Q_(1.00, ureg.mol)
T = Q_(58.8, ureg.degC).to(ureg.K)

q_rev = Delta_H * n
Delta_S = (q_rev / T).to(ureg.J / ureg.K)

display(Markdown(fr"1.00 mol $\mathrm{{Br}}_{{2}}(l)$ 在 $58.8\;^{{\circ}}\mathrm{{C}}$ 汽化时的 $\Delta S$ 值为 {Delta_S.magnitude:.3g} J/K。"))

1.00 mol $\mathrm{Br}_{2}(l)$ 在 $58.8\;^{\circ}\mathrm{C}$ 汽化时的 $\Delta S$ 值为 89.2 J/K。

---

> **点评人： GPT-5.5**
>
> 整体评价：答案正确，代码计算也给出了合理数值。这个题的关键是认识到在正常沸点下，液态溴与溴蒸气处于相平衡，汽化可按可逆等温相变处理，因此可以直接用 $\Delta S = q_{\mathrm{rev}}/T$。
>
> - **(a)** 正确。液体汽化为气体时，分子可占据的空间和微观状态数显著增加，因此系统熵增加，即 $\Delta S > 0$。
> - **(b)** 思路正确。1.00 mol $\mathrm{Br}_{2}(l)$ 在正常沸点汽化时，$q_{\mathrm{rev}} = n\Delta H_{vap}$，温度应使用开尔文：$T = 58.8 + 273.15 = 331.95\ \mathrm{K}$。因此
>
> $$
> \Delta S = \frac{(1.00\ \mathrm{mol})(29.6\times 10^{3}\ \mathrm{J/mol})}{331.95\ \mathrm{K}} = 89.2\ \mathrm{J/K}
> $$
>
> 你的计算结果正确。书写答案时建议把 $\Delta H_{vap}$ 从 kJ/mol 转成 J/mol，并明确温度必须用 K，这样单位会自然得到 $\mathrm{J/K}$。
:::

::: {.callout-tip title="习题 19.28"}
（a）自发过程中宇宙的熵在何时增加？（b）在某自发过程中，系统熵增加。关于 $\Delta S_{surr}$ 的符号和大小，你能得出什么结论？（c）在某可逆过程中，环境熵变为 $\Delta S_{surr} = -78 J/K$。该过程中系统的熵变是多少？

**解：**

**(a)** 系统熵和环境熵变化量之和小于 0 的时候。

**(b)** 自发过程中，$\Delta S_{univ} = \Delta S_{sys} + \Delta S_{surr} > 0$，题目又提出 $\Delta S_{sys} > 0$，那么 $\Delta S_{surr}$ 可能有多种情况：

- $\Delta S_{surr} \geq 0$
- $\Delta S_{surr} \lt 0, -\Delta S_{surr} < \Delta S_{sys}$

**(c)** 题目指明为可逆过程，此时有 $\Delta S_{univ} = \Delta S_{sys} + \Delta S_{surr} = 0$。因此 $\Delta S_{sys} = - \Delta S_{surr} = 78 J/K$

---

> **点评人： GPT-5.5**
>
> 整体评价：你的 (b) 和 (c) 思路很好，已经能正确使用“宇宙熵变 = 系统熵变 + 环境熵变”这一判据。需要重点修正的是 **(a)**：自发过程对应的是宇宙熵增加，而不是小于 0。
>
> - **(a)** 这里不等号方向写反了。自发过程是不可逆过程，因此根据热力学第二定律，$\Delta S_{\mathrm{univ}} = \Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}} > 0$。若为可逆过程，则 $\Delta S_{\mathrm{univ}} = 0$。
> - **(b)** 正确。已知 $\Delta S_{\mathrm{sys}} > 0$ 且过程自发，所以只要求 $\Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}} > 0$。因此 $\Delta S_{\mathrm{surr}}$ 可以为正、为零，也可以为负；若为负，其绝对值必须小于系统熵增，即 $|\Delta S_{\mathrm{surr}}| < \Delta S_{\mathrm{sys}}$。
> - **(c)** 正确。可逆过程满足 $\Delta S_{\mathrm{univ}} = 0$，所以 $\Delta S_{\mathrm{sys}} = -\Delta S_{\mathrm{surr}} = 78\ \mathrm{J/K}$。
>
> 另外，符号书写上建议与本节教材统一，使用 $\Delta S_{\mathrm{univ}}$、$\Delta S_{\mathrm{sys}}$、$\Delta S_{\mathrm{surr}}$，这样下标会以正体显示，更像标准热力学记号。
:::

::: {.callout-tip title="习题 19.29"}
（a）0.200 mol 理想气体在 $27^{\circ}C$ 下等温膨胀，初始体积为 10.0 L，你预期 $\Delta S$ 的符号是什么？（b）若最终体积为 18.5 L，计算该过程的熵变。（c）计算熵变是否需要指定温度？请解释。

**解：**

**(a)** 理想气体方程为 $p V = n R T$。其中 $R$ 为物理常数，$n$ 和 $T$ 在本题中为固定值。又已知气体膨胀，所以气压下降。整个物理变化过程中，微观状态数增加，预期 $\Delta S \gt 0$。

**(b)** 理想气体等温可逆过程中，有：

- $\Delta E = q_{\mathrm{rev}} + w = 0$ 或者说 $q_{\mathrm{rev}} = -w$
- $\Delta S = q_{rev} / T$
- $\displaystyle w = -\int_{V_1}^{V_2} p(V) \mathrm{d} V = -\int_{V_1}^{V_2} \frac{n R T}{V} \mathrm{d} V = -nRT \ln \frac{V_{2}}{V_{1}}$

因此

$$
\Delta S = \frac{-w}{T} = nR \ln \frac{V_{2}}{V_{1}}
$$


In [64]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

n = Q_(0.200, ureg.mol)
V_2 = Q_(18.5, ureg.L)
V_1 = Q_(10.0, ureg.L)
R = Q_(8.314, ureg.J / ureg.mol / ureg.K)

Delta_S = n * R * np.log((V_2 / V_1).to(ureg.dimensionless))
Delta_S = Delta_S.to(ureg.J / ureg.K)
display(Markdown(fr"该过程的熵变为 {Delta_S.magnitude:.3g} J/K"))

该过程的熵变为 1.02 J/K

**(c)** 不需要指定温度，(b) 中的推导和代码可以看出。

---

> **点评人： GPT-5.5**
>
> 整体评价：这道题完成得很好，尤其是 (b) 中通过可逆等温路径推导 $\Delta S = nR\ln(V_{2}/V_{1})$，再用 Python 计算，思路和数值都正确。这里的关键概念是：熵是状态函数，所以即使实际膨胀过程未必可逆，也可以选取一条连接同一初末态的可逆路径来计算 $\Delta S$。
>
> - **(a)** 正确。等温膨胀时体积增大，气体分子可占据的空间增加，微观状态数增加，因此 $\Delta S > 0$。你从微观状态数增加来判断非常合适；气压下降只是伴随现象，不是判断熵变符号的核心依据。
> - **(b)** 正确。对理想气体等温可逆膨胀，$w_{\mathrm{rev}} = -nRT\ln(V_{2}/V_{1})$，因此 $q_{\mathrm{rev}} = -w_{\mathrm{rev}} = nRT\ln(V_{2}/V_{1})$，代入 $\Delta S = q_{\mathrm{rev}}/T$ 得 $\Delta S = nR\ln(V_{2}/V_{1})$。数值为 $0.200\times 8.314\times \ln(18.5/10.0) = 1.02\ \mathrm{J/K}$，与你的代码输出一致。
> - **(c)** 结论正确。给定 $n$、$V_{1}$、$V_{2}$ 后，理想气体等温膨胀的熵变公式中温度会抵消，因此不需要用题目给出的 $27\ ^\circ\mathrm{C}$ 来计算数值。不过“等温”这个条件仍然重要；如果不是等温过程，计算路径和公式就会不同。
>
> 书写上建议把 $q_{rev}$ 统一写成 $q_{\mathrm{rev}}$，把功写成 $w_{\mathrm{rev}}$，这样更清楚地表明你是在使用可逆路径计算熵变。
:::


## 第 3 节 熵的分子解释与热力学第三定律

宏观热力学中的熵，可在分子层面理解为系统**随机性**或**能量分散程度**的量度。Ludwig Boltzmann 把熵与微观状态数联系起来，为第二定律提供了分子解释。

### 气体膨胀的分子图景

气体向更大体积自发扩散，是因为分子可及排布方式急剧增多。例如两个分子原本局限在一侧烧瓶中，打开活塞后可在两侧自由分布，微观排布从 1 种变为 4 种；一摩尔气体（$N \approx 6.02 \times 10^{23}$）全部回到原侧的概率为 $(1/2)^N$，实质上为零。因此气体倾向于占据全部可用空间——最可能的排布是两个容器中分子数大致相等。

### Boltzmann 方程与微观状态

**宏观状态**由温度、压强、体积等描述；**微观状态**是分子位置与动能的一种具体排布。给定宏观状态下，所有可能微观状态的数目记为 $W$。Boltzmann 方程为

$$
S = k \ln W
$$

其中 $k = 1.38 \times 10^{-23}\ \mathrm{J/K}$ 为 Boltzmann 常数。过程的熵变为

$$
\Delta S = k \ln \frac{W_{\text{final}}}{W_{\text{initial}}}
$$

$W_{\text{final}} > W_{\text{initial}}$ 时 $\Delta S > 0$。通常，**体积增大**、**温度升高**或**分子数/分子复杂度增加**，都会使可及微观状态数增多，熵随之增加——这与第 19.2 节理想气体等温膨胀 $\Delta S = nR \ln(V_2/V_1)$ 的结论一致。

### 分子运动方式

真实分子除平动外，还可**振动**（键长周期性变化）和**转动**（绕轴旋转）。气体分子平动自由度最大，液体次之，固体最小（以振动为主）。分子越复杂，可能的振动方式越多，微观状态数通常也越多。

### 对 $\Delta S$ 作定性预测

以水的三相为例：冰中氢键形成刚性晶格，运动以振动为主；液态水可平动、转动；气态水分子彼此独立、运动完全自由。故 $S(\text{冰}) < S(\text{液}) < S(\text{气})$。`习题 19.5` 的熵-温度图正是这一思想的宏观体现。

一般预期下列过程中系统熵增加：

1. 由固体或液体生成气体；
2. 由固体生成液体或溶液；
3. 化学反应中气体分子数增加。

反之，气体分子数减少或形成更有序固体，往往 $\Delta S < 0$。`习题 19.33`、`习题 19.41`、`习题 19.43` 都建立在这一套定性判断之上。

### 热力学第三定律

纯物质、**完美晶体**在绝对零度（$0\ \mathrm{K}$）时熵为零：

$$
S(0\ \mathrm{K}) = 0
$$

此时粒子占据完全有序的晶格位置、无热运动，仅对应一种微观状态（$W = 1$，故 $S = k \ln 1 = 0$）。温度从 0 K 升高时，振动等自由度逐渐激活，熵增加。第三定律为第 19.4 节中由表列 $S^{\circ}$ 计算反应 $\Delta S^{\circ}$ 提供了熵的零点基准。

::: {.callout-tip title="习题 19.5"}
附图显示了某物质的熵随温度如何变化，该物质在所示最高温度下为气体。（a）标有 1 和 2 的竖直线上熵的增加分别对应哪些过程？（b）为何 2 的熵变大于 1？（c）若该物质在 $T = 0$ K 时为完美晶体，此温度下 $S$ 的值是多少？[第 19.3 节]
![](./assets/images/8f67edd267626dbb7da2a5c2b13e7322a0e648e59a6b956785b8ec6523a7f715.jpg){fig-align="center"}

温度 (K)

**解：** 

**(a)** 1 对应溶化，2 对应汽化。

**(b)** 熵变与微观状态数正相关。物质从固体相变为液体，在振动之上，增加了平动、转动自由度。从液体相变为气体，空间排列组合情况增加。空间变大提供的微观状态数贡献大于自由度变多提供的，所以 2 的熵变大于 1。

**(c)** 绝对零度的情况下，需要考虑物质的几何结构才能够判断熵是否为 0。完美晶体意味着它的几何结构应该是完美对称的，所以熵为 0。

---

> **点评人： GPT-5.5**
>
> 整体评价：三个结论都抓住了题目的主线，尤其是把竖直跃迁识别为相变，并指出汽化的熵变大于熔化的熵变，这是本题最重要的判断。
>
> - **(a)** 正确。曲线中标号 1 对应固体变为液体，即熔化；标号 2 对应液体变为气体，即汽化。若用术语写得更规范，可写作“熔化”和“汽化”，而不是“溶化”。
> - **(b)** 判断正确。两次竖直上升都是等温相变时的熵突增；汽化时分子从液体进入气体，相比熔化时从固体进入液体，分子可占据的体积和位置分布大得多，因此微观状态数增加得更多，$\Delta S_{vap} > \Delta S_{fus}$。你用“空间排列组合显著增加”来解释 2 更大，是很好的直觉。
> - **(c)** 结论正确，但理由可以更贴近热力学第三定律：完美晶体指所有粒子都占据规则晶格中的确定位置、没有缺陷、没有取向无序或构型无序的理想晶体。这样的晶体在 $T = 0\ \mathrm{K}$ 时只有一种可能的排列方式，即 $\Omega = 1$，因此 $S = k\ln\Omega = 0$。关键不是一般意义上的“几何结构对称”，而是“完美有序、唯一微观态”。
>
> 小建议：这类熵图中，斜线段表示同一相内随温度升高熵逐渐增加，竖直跃迁表示相变带来的熵突增。这样读图会更稳定。
:::


::: {.callout-tip title="习题 19.33"}
下列变化会使系统可用的微观态数目增加、减少还是不变：（a）温度升高，（b）体积减小，（c）由液态变为气态？

**解：**

**(a)** 温度升高，分子热运动加剧，速度分布曲线更宽更扁。如果温度升高的过程中，系统中有物质发生相变，那么微观状态数一定增加。

如果没有物质相变呢？考虑理想气体，它基于分子无体积，分子间无任何相互作用。这种情况下，我认为只要系统的体积不变，那么分子的排列组合就不会有变化，所以微观状态数不变。尽管分子热运动加剧。

还需要单独考虑固体和液体吗？

**(b)** 体积变小，分子能够去的地方变少，微观状态数自然减少。

**(c)** 汽化，微观状态数增加。

---

> **点评人： GPT-5.5**
>
> 整体评价：你对 **(b)** 和 **(c)** 的判断很准确，解释也抓住了“可访问空间”与“相态变化”这两个直观因素。需要重点修正的是 **(a)**：温度升高本身就会使系统可用的微观态数目增加，即使没有发生相变、体积也不变。
>
> - **(a)** 应为**增加**。微观态不只包括粒子“在空间中怎么排”，也包括能量如何分配到平动、转动、振动等运动方式中。温度升高时，系统可占据更高能级，能量分布方式增多，因此可用微观态数目增加。理想气体即使体积不变，速度/动能分布的可能性也变多，所以不是不变。固体和液体也类似：温度升高会增加振动能级等可访问状态。
> - **(b)** 正确。体积减小时，粒子可占据的位置范围变小，位置相关的微观态数减少，因此系统可用微观态数目减少。
> - **(c)** 正确。液态变为气态时，粒子之间约束显著减弱、可占据空间大幅增加，微观态数目增加。
>
> 这题可以用一句话记：升温增加“能量分布”的可能性，增大体积或变成气体增加“空间分布”的可能性。
:::


::: {.callout-tip title="习题 19.41"}
在下列各对物质中，选出给定温度下每摩尔熵较高者：（a）$\mathrm{Ne}(l)$ 或 $\mathrm{Ne}(g)$，（b）5 atm 下的 $\mathrm{H}_{2}(g)$ 或 3 atm 下的 $\mathrm{H}_{2}(g)$，（c）10.0 L 中的 1 mol $\mathrm{Kr}(g)$ 或 12.0 L 中的 5 mol $\mathrm{Kr}(g)$，（d）$\mathrm{NO}_{2}(g)$ 或 $\mathrm{NO}_{2}(s)$。

**解：**

**(a)** $\mathrm{Ne}(g)$，显然气体比液体更加无序。

**(b)** 根据 $p V = n R T$，p 增大，n R T 保持不变，那么 V 减少。相空间告诉我们，一个系统的某一个微观状态由 $x, y, z, p_x, p_y, p_z$ 决定。对于理想气体，其内能与温度相关。温度相同的情况下，各个气体分子能够分配到的 $p_x, p_y, p_z$ 概率相同，因此只考虑 $x, y, z$ 空间排列。5 atm 下氢气占据的体积比 3 atm 的小，所以前者微观状态数小于后者。因此 3 atm 下的 $\mathrm{H}_{2}(g)$ 的每摩尔熵更高。

**(c)** 12.0 L 中的 5 mol $\mathrm{Kr}(g)$。因为考虑的是**每摩尔**熵，所以不管物质的量，选择空间更大者即可。

**(d)** $\mathrm{NO}_{2}(g)$。气体比固体更无序。

---

> **点评人： GPT-5.5**
>
> 整体评价：你对相态和压强影响熵的直觉很好，(a)、(b)、(d) 都判断正确；需要重点修正的是 (c)，这里“每摩尔熵”要求比较的是每 1 mol 气体可占据的体积，而不是容器总体积。
>
> - **(a)** 正确。相同物质、相同温度下，气态比液态有更多可访问位置和运动状态，因此 $\mathrm{Ne}(g)$ 的摩尔熵更高。
> - **(b)** 正确。相同温度下，压强较低意味着气体摩尔体积更大，分子可占据的位置更多，所以 3 atm 下的 $\mathrm{H}_{2}(g)$ 每摩尔熵更高。你用相空间的角度解释很有深度。
> - **(c)** 这里应选 **10.0 L 中的 1 mol $\mathrm{Kr}(g)$**。因为题目问的是“每摩尔熵”，应比较摩尔体积 $V/n$：前者为 $10.0\ \mathrm{L/mol}$，后者为 $12.0/5 = 2.4\ \mathrm{L/mol}$。前者每摩尔气体拥有更大的空间，因此摩尔熵更高。
>   *通俗点说：虽然 12.0 L 的“总房子”比 10.0 L 大，但里面挤了整整 5 摩尔的分子；而 10.0 L 的房子里只有 1 摩尔分子独享。因为我们要比较的是“每摩尔熵”（即平均每个分子的混乱度），所以必须看平均生存空间。10.0 L/mol 远比 2.4 L/mol 宽敞，微观状态数更多。*
> - **(d)** 正确。气态 $\mathrm{NO}_{2}$ 比固态 $\mathrm{NO}_{2}$ 有更多平动自由度和空间分布方式，因此摩尔熵更高。
>
> 小结：比较气体摩尔熵时，若温度相同，常看“每摩尔可用空间”或摩尔体积；不是简单看总容器体积。
:::


::: {.callout-tip title="习题 19.43"}
预测下列各反应中系统熵变的符号：

(a) $\mathrm{N}_{2}(g) + 3\mathrm{H}_{2}(g) \longrightarrow 2\mathrm{NH}_{3}(g)$

(b) $\mathrm{CaCO_{3}(s)} \longrightarrow \mathrm{CaO(s)} + \mathrm{CO_{2}(g)}$

(c) $3\mathrm{C}_{2}\mathrm{H}_{2}(g)\longrightarrow\mathrm{C}_{6}\mathrm{H}_{6}(g)$

(d) $\mathrm{Al_{2}O_{3}(s)} + 3\mathrm{H_{2}(g)} \longrightarrow 2\mathrm{Al(s)} + 3\mathrm{H_{2}O(g)}$

**解：**

**(a)** 负号。反应物与生成物均为气态，可应用理想气体公式 $p V = n R T$。反应向右进行过程中，$n$ 下降，因此 $V$ 下降，进而微观状态数减少，该反应的熵变为负号。

**(b)** 正号。生成物存在气体，所以更加无序。

**(c)** 负号。理由与 (a) 相同。

**(d)** 正号。反应物和生成物均有气态物质。可以指考虑气体之间的熵变程度。考虑系数，$\mathrm{H_{2}(g)}$ 和 $\mathrm{H_{2}O(g)}$ 系数相同。考虑运动自由度，$\mathrm{H_{2}(g)}$ 的化学键只有一组，而 $\mathrm{H_{2}O(g)}$ 有两组，所以后者的振动与转动的自由度更丰富，故反应向右进行，整个系统将会更加无序。

---

> **点评人： Composer 2.5**
>
> 整体评价：四道题的 $\Delta S$ 符号全部判断正确！你对 (a)(c) 用气态物质的量变化来推断熵变，思路清晰；(d) 在气体摩尔数不变时进一步比较分子复杂度，已经超出了“只数气体分子”的入门层次，值得肯定。
>
> - **(a)** 正确。反应物侧气态物质的量为 $1 + 3 = 4\ \mathrm{mol}$，生成物侧为 $2\ \mathrm{mol}$，$\Delta n_{\mathrm{gas}} = -2 < 0$，故 $\Delta S < 0$。你用 $pV = nRT$ 把“物质的量减少”与“体积收缩、微观状态数减少”联系起来，解释合理。
> - **(b)** 正确。反应由两种固体生成一种固体加一种气体，关键是产生了 $\mathrm{CO_2(g)}$，气态分子数从 0 增至 1，系统无序度显著增加，故 $\Delta S > 0$。可再补一句：固体熵值通常远小于气体，因此“生成气体”往往是 $\Delta S > 0$ 的决定性因素。
> - **(c)** 正确。$\Delta n_{\mathrm{gas}} = 1 - 3 = -2$，三个气态乙炔分子缩合成一个苯分子，可用空间与平动自由度均减少，$\Delta S < 0$。与 (a) 同属“气体摩尔数减少 → 熵减”这一类。
> - **(d)** 正确，且理由很有深度。两侧气态物质的量均为 $3\ \mathrm{mol}$，若只看 $\Delta n_{\mathrm{gas}}$ 会误判为“约等于零”；你注意到 $\mathrm{H_2O(g)}$ 为非线性三原子分子，比双原子 $\mathrm{H_2(g)}$ 拥有更多转动与振动模式，单摩尔熵更高，因此 $\Delta S > 0$。小建议：文中“可以指考虑”应为“可以只考虑”（笔误）。
>
> 小结：预测反应熵变符号时，优先比较 $\Delta n_{\mathrm{gas}}$；当气态摩尔数相同时，再比较气体分子的复杂度（原子数、结构、自由度）。固体若两侧都有且变化不显著，在符号判断中通常可忽略。
:::


## 第 4 节 化学反应中的熵变

与 $\Delta H$ 不同，化学反应的 $\Delta S$ 没有与之相当的量热法可直接测量。但热力学第三定律为熵确立了零点，因此可以通过实验测定物质的绝对熵 $S$，进而由表列数据计算反应的标准熵变。

纯物质、完美晶体在 $0\ \mathrm{K}$ 时熵为零；温度升高时熵稳步增加。固体在熔点、液体在沸点处，因分子获得新的自由度或可占据体积显著增大，熵出现陡增。熵-温度图（如 `习题 19.5` 附图）中，斜线段表示同一相内随升温熵增，竖直跃迁表示相变带来的熵突增。

**标准摩尔熵** $S^{\circ}$ 指标准状态下（纯物质、$1\ \mathrm{atm}$）物质每摩尔的熵，单位 $\mathrm{J/(mol\cdot K)}$，298 K 下的值见附录 C。与标准生成焓不同，元素在 298 K 的 $S^{\circ}$ **不为零**。比较 $S^{\circ}$ 时有几点规律：

1. 气态 $>$ 液态 $>$ 固态；
2. 通常随摩尔质量增大而增加；
3. 通常随分子中原子数增多而增加（自由度、可及微观态更多）。

化学反应的标准熵变等于生成物标准熵之和减去反应物标准熵之和：

$$
\Delta S^{\circ} = \sum n S^{\circ}(\text{products}) - \sum m S^{\circ}(\text{reactants})
$$

其中 $n$、$m$ 为平衡化学方程式中的化学计量数。计算时查附录 C 中各物种的 $S^{\circ}$，再按系数加权求和——这正是 `习题 19.49`、`习题 19.53` 所要求的方法。预判 $\Delta S^{\circ}$ 符号时，可先看气态物质的量是否变化（$\Delta n_{\mathrm{gas}}$），以及是否产生气体；但最终仍应以查表计算为准。

### 环境的熵变

系统发生反应时，环境可视为巨大的恒温热源（或热汇）。对等温过程，环境熵变为

$$
\Delta S_{\mathrm{surr}} = \frac{-q_{\mathrm{sys}}}{T}
$$

恒压过程中 $q_{\mathrm{sys}} = \Delta H_{\mathrm{sys}}$，故

$$
\Delta S_{\mathrm{surr}} = \frac{-\Delta H_{\mathrm{sys}}}{T} \quad [\text{恒压}]
$$

宇宙（系统 + 环境）的总熵变为

$$
\Delta S_{\mathrm{univ}} = \Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}}
$$

放热反应中系统向环境放热（$q_{\mathrm{sys}} < 0$），环境吸热、熵增；即使系统本身熵减（如合成氨 $\Delta S^{\circ}_{\mathrm{sys}} < 0$），环境熵增仍可能使 $\Delta S_{\mathrm{univ}} > 0$，从而驱动反应自发进行。须注意：热力学判据只说明过程是否自发，并不决定反应速率。

::: {.callout-tip title="习题 19.6"}
同分异构体是具有相同化学式但原子排列不同的分子，如图所示为戊烷（$C_{5}H_{12}$）的两种同分异构体。（a）你预期两种同分异构体的燃烧焓会有显著差异吗？请解释。（b）你预期哪种同分异构体的标准摩尔熵更高？请解释。[第 19.4 节]
![](./assets/images/0350477392ed3ae9b7880ca249403ebc97529edfa94773005cc6a6e451a0af8d.jpg){fig-align="center"}

正戊烷

新戊烷

**解：**

**(a)** 两者碳氢键与碳碳键的数量相同，基于此我推测二者的燃烧焓不会有显著差异。

**(b)** 我预期前者的标准摩尔熵更高。前者的四个碳碳键具有更多的旋转自由度，想象如果转动前者的任意碳碳键，整个分子的几何结构将发生显著变化。

---

> **点评人： Composer 2.5**
>
> 整体评价：结合配图判断，两问结论均正确。你已能从“键的类型与数目”和“分子柔性/构象”两个角度分别讨论焓与熵，思路与教材 19.4 节的意图一致。
>
> - **配图确认**：图中左侧为**正戊烷**（$n$-pentane，$CH_3CH_2CH_2CH_2CH_3$，直链锯齿形），右侧为**新戊烷**（neopentane，$C(CH_3)_4$，中心碳连四个甲基，结构紧凑、近似球形）。二者化学式同为 $C_5H_{12}$，互为同分异构体。
> - **(a)** 正确。同分异构体具有相同的分子式，完全燃烧后均生成相同物质的量的 $\mathrm{CO_2(g)}$ 和 $\mathrm{H_2O(l)}$，总反应相同，因此燃烧焓**不会有显著差异**。更严格地说：C–C、C–H 键的总数相同， Hess 定律下 $\Delta H$ 应相近；实际数据中正戊烷与新戊烷的标准燃烧焓仅相差约 $5\ \mathrm{kJ/mol}$ 量级（在 $\sim 3500\ \mathrm{kJ/mol}$ 中占比极小），差异来自支链结构带来的稳定性不同（新戊烷略更稳定），但远称不上“显著”。
> - **(b)** 正确，应选**正戊烷**（左侧）的标准摩尔熵更高。直链分子绕 C–C 单键可形成多种构象（如交叉式、邻位交叉式等），可及微观状态数多，构象熵大；新戊烷支链高度紧凑、对称性高，整体刚性更强，可及构象远少于长链分子，故 $S^\circ$ 略低（298 K 下正戊烷 $S^\circ \approx 263.5\ \mathrm{J/(mol\cdot K)}$，新戊烷 $S^\circ \approx 260.4\ \mathrm{J/(mol\cdot K)}$）。你用“C–C 键旋转改变整体几何”来理解正戊烷的柔性，方向很好；可再补一句：比较同分异构体摩尔熵时，**链长与支化程度**是关键——直链、柔性大者通常熵更高。
>
> 小结：同分异构体常“焓相近、熵有差”——键数相同使燃烧焓接近，而分子形状与构象自由度决定熵的细微差别。
:::


::: {.callout-tip title="习题 19.48"}
环丙烷与丙烯是同分异构体，化学式均为 $C_{3}H_{6}$。根据所示分子结构，在 $25^{\circ}C$ 下你预期哪种同分异构体的标准摩尔熵更高？

![](./assets/images/38ec3d22d0904024df6c83a9f9330dc10ba7797ca61ac18eb9eeb187ed634ad4.jpg){fig-align="center"}

环丙烷

![](./assets/images/7e28841ccb94f4b011a883d7f556fdbd5470cc15a53f87675c311cd3498b6d9c.jpg){fig-align="center"}

丙烯

**解：**

我认为丙烯的标准摩尔熵更高。环丙烷的三个碳原子彼此相连构成了三角形，具有稳定性；丙烯的三个碳原子为直链结构。故后者熵更高。

---

> **点评人： Composer 2.5**
>
> 整体评价：**结论正确**——丙烯的标准摩尔熵高于环丙烷。但推理中有两处需要修正：不宜用“稳定性”来解释熵的高低，且丙烯并非简单的“直链”结构。
>
> - **配图确认**：第一张图为**环丙烷**（三元碳环，每个 C 连 2 个 H）；第二张图为**丙烯**（$CH_2=CH-CH_3$，含 $C=C$ 双键与末端甲基）。二者化学式均为 $C_3H_6$。
> - **结论** 正确。298 K 下近似有 $S^\circ(\text{环丙烷}) \approx 184.5\ \mathrm{J/(mol\cdot K)}$，$S^\circ(\text{丙烯}) \approx 197.9\ \mathrm{J/(mol\cdot K)}$，丙烯更高。
> - **推理修正 1**：环丙烷“具有稳定性”这一表述容易误导。三元环实际上存在显著的**环张力**（键角偏离正常 $109.5^\circ$），在热力学上比丙烯**更不稳定**（这主要属于焓的讨论）。比较熵时应强调：环丙烷是**刚性小环**，整体构象几乎被“锁死”，可及微观状态数少，故 $S^\circ$ 较低。
> - **推理修正 2**：丙烯不是丙烷那样的直链烷烃，而是**开链烯烃**。虽然 $C=C$ 处不能自由旋转，但 $C–C$ 单键（双键与甲基之间）及甲基仍可转动，整体柔性大于环丙烷，构象熵更大，因此 $S^\circ$ 更高。
>
> 小结：同分异构体比摩尔熵时，**环状 vs 开链**往往是关键——小环刚性大、构象受限 → 熵低；开链分子可及构象更多 → 熵高。这与 19.6 中“分子柔性决定熵”的思路一脉相承。
:::


::: {.callout-tip title="习题 19.49"}
利用附录 C，比较下列各对物质在 25°C 下的标准熵：（a）Sc(s) 与 Sc(g)，（b）NH₃(g) 与 NH₃(aq)，（c）1 mol P₄(g) 与 2 mol P₂(g)，（d）C（石墨）与 C（金刚石）。

**解：**

**(a)** Scandium。Sc(s) 固态标准熵为 34.6，Sc(g) 气态为 174.7。单位均为 J/mol-K

同一物质，其气态标准熵大于固态。

**(b)** NH₃(aq) 111.3 J/mol-K 和 NH₃(g) 192.5 J/mol-K

同一物质，其气态标准熵大于液态。

**(c)** 标准熵不考虑物质的量，从单位上也能看得出来。280 J/mol-K

相同形态的相同物质的标准熵相同。

**(d)**  C（石墨）5.69； C（金刚石）2.43 J/mol-K

金刚石更稳定、有序，所以标准熵更低。

---

> **点评人： Composer 2.5**
>
> 整体评价：(a)(b)(d) 查表数值与大小关系基本正确；(c) 需要重点修正——题目比较的是 **1 mol $\mathrm{P_4(g)}$ 与 2 mol $\mathrm{P_2(g)}$ 的总熵**，不能只说“摩尔熵与物质的量无关”就结束。
>
> - **(a)** 正确。附录 C：$\ce{Sc(s)}$ $S^\circ = 34.6\ \mathrm{J/(mol\cdot K)}$，$\ce{Sc(g)}$ $S^\circ = 174.7\ \mathrm{J/(mol\cdot K)}$，气态更高。同一元素，气相比固相分子运动与空间分布更自由，微观状态数更多。
> - **(b)** 数值正确：$\ce{NH3(g)}$ $192.5\ \mathrm{J/(mol\cdot K)}$，$\ce{NH3(aq)}$ $111.3\ \mathrm{J/(mol\cdot K)}$，气态更高。术语需微调：$\ce{NH3(aq)}$ 是**水溶液中**的氨，不是液态氨 $\ce{NH3(l)}$；氨分子溶于水后运动受溶剂限制，熵低于气态。
> - **(c)** **需修正**。附录 C：$\ce{P4(g)}$ $S^\circ = 280\ \mathrm{J/(mol\cdot K)}$，$\ce{P2(g)}$ $S^\circ = 218.1\ \mathrm{J/(mol\cdot K)}$。你说“标准熵不考虑物质的量”只对了一半——$S^\circ$ 是**每摩尔物质**的熵，但本题给的是 **1 mol $\ce{P4}$ 与 2 mol $\ce{P2}$**（均含 4 个 P 原子），应比较**总熵**：
>   $$S_{\text{total}}(1\ \mathrm{mol\ P_4}) = 280\ \mathrm{J/K}$$
>   $$S_{\text{total}}(2\ \mathrm{mol\ P_2}) = 2 \times 218.1 = 436.2\ \mathrm{J/K}$$
>   故 **2 mol $\ce{P2(g)}$ 的标准熵更高**。原因：分子数更多（2 mol vs 1 mol 气体分子），且 $\ce{P4}$ 为四面体单分子、结构紧凑，$\ce{P2}$ 为两个独立双原子分子，可及微观状态更多。
> - **(d)** 数值与大小关系正确：石墨 $5.69 >$ 金刚石 $2.43\ \mathrm{J/(mol\cdot K)}$。金刚石三维网络刚性极强、振动模式受限，熵更低——“更有序 → 熵低”说得对。但“金刚石更稳定”在 25 °C、1 atm 下**不成立**：该条件下**石墨**是碳的热力学稳定相，金刚石为亚稳态；稳定性属于焓/自由能讨论，与本次熵比较应分开表述。
>
> 小结：查 $S^\circ$ 时注意相态标注；若题目指定物质的量（如 2 mol），需用 $S_{\text{total}} = n \cdot S^\circ$ 比较总熵。
:::


::: {.callout-tip title="习题 19.53"}
利用附录 C 中的 $S^{\circ}$ 值，计算下列反应的 $\Delta S^{\circ}$ 值。每种情况下解释 $\Delta S^{\circ}$ 符号的原因。

(a) $\mathrm{C_{2}H_{4}(g) + H_{2}(g)} \longrightarrow \mathrm{C_{2}H_{6}(g)}$

(b) $\mathrm{N_{2}O_{4}(g)} \longrightarrow 2\mathrm{NO_{2}(g)}$

(c) $\mathrm{Be(OH)_{2}(s)} \longrightarrow \mathrm{BeO(s) + H_{2}O(g)}$

(d) $2\mathrm{CH_{3}OH}(g) + 3\mathrm{O_{2}(g)} \longrightarrow 2\mathrm{CO_{2}(g)} + 4\mathrm{H_{2}O(g)}$

**解：**

**(a)** 反应物和生成物均为气相。生成物摩尔的量之和小于反应物的，预期 $\Delta S^{\circ} \lt 0$，系统微观状态数减少。

In [65]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

# C2H4(g)
S_1 = Q_(219.4, ureg.J / ureg.mol / ureg.K)
n_1 = Q_(1, ureg.mol)
# H2(g)
S_2 = Q_(130.58, ureg.J / ureg.mol / ureg.K)
n_2 = Q_(1, ureg.mol)
# C2H6(g)
S_3 = Q_(229.5, ureg.J / ureg.mol / ureg.K)
n_3 = Q_(1, ureg.mol)

Delta_S = S_3 * n_3 - (S_1 * n_1 + S_2 * n_2)
Delta_S = Delta_S.to(ureg.J / ureg.K)
display(Markdown(fr"$\Delta S^{{\circ}}$ = {Delta_S.magnitude:.3g} J/K"))

$\Delta S^{\circ}$ = -120 J/K

计算结果符合预期。

**(b)** 先做预测。均为气相，生成物摩尔的量更大，$\Delta S^{\circ} \gt 0$，更无序。

In [66]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

# N2O4(g)
S_1 = Q_(304.3, ureg.J / ureg.mol / ureg.K)
n_1 = Q_(1, ureg.mol)
# NO2(g)
S_2 = Q_(240.45, ureg.J / ureg.mol / ureg.K)
n_2 = Q_(2, ureg.mol)

Delta_S = S_2 * n_2 - S_1 * n_1
Delta_S = Delta_S.to(ureg.J / ureg.K)
display(Markdown(fr"$\Delta S^{{\circ}}$ = {Delta_S.magnitude:.3g} J/K"))

$\Delta S^{\circ}$ = 177 J/K

计算结果符合预期。

**(c)** 生成物出现了气相 $\ce{H2O(g)}$。通常，气相物质的熵远远大于固相的。因此预期 $\Delta S^{\circ} \gt 0$，更无序。

In [67]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

# Be(OH)2(s)
S_1 = Q_(50.21, ureg.J / ureg.mol / ureg.K)
n_1 = Q_(1, ureg.mol)
# BeO(s)
S_2 = Q_(13.77, ureg.J / ureg.mol / ureg.K)
n_2 = Q_(1, ureg.mol)
# H2O(g)
S_3 = Q_(188.83, ureg.J / ureg.mol / ureg.K)
n_3 = Q_(1, ureg.mol)

Delta_S = (S_2 * n_2 + S_3 * n_3) - S_1 * n_1
Delta_S = Delta_S.to(ureg.J / ureg.K)
display(Markdown(fr"$\Delta S^{{\circ}}$ = {Delta_S.magnitude:.3g} J/K"))

$\Delta S^{\circ}$ = 152 J/K

计算结果符合预期。

**(d)** 生成物摩尔的量为 6 mol，反应物为 5 mol。预期 $\Delta S^{\circ} \gt 0$。


In [68]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

# CH3OH(g)
S_1 = Q_(237.6, ureg.J / ureg.mol / ureg.K)
n_1 = Q_(2, ureg.mol)
# O2(g)
S_2 = Q_(205.0, ureg.J / ureg.mol / ureg.K)
n_2 = Q_(3, ureg.mol)
# CO2(g)
S_3 = Q_(213.6, ureg.J / ureg.mol / ureg.K)
n_3 = Q_(2, ureg.mol)
# H2O(g)
S_4 = Q_(188.83, ureg.J / ureg.mol / ureg.K)
n_4 = Q_(4, ureg.mol)

Delta_S = (S_3 * n_3 + S_4 * n_4) - (S_1 * n_1 + S_2 * n_2)
Delta_S = Delta_S.to(ureg.J / ureg.K)
display(Markdown(fr"$\Delta S^{{\circ}}$ = {Delta_S.magnitude:.3g} J/K"))

$\Delta S^{\circ}$ = 92.3 J/K

计算结果符合预期。

---

> **点评人： Composer 2.5**
>
> 整体评价：四道题的符号判断、附录 C 查表与 pint 计算全部正确；“先预测符号 → 再算 $\Delta S^\circ$ → 核对是否符合预期”的流程非常规范，正是 19.4 节所要求的思路。
>
> - **(a)** 正确。$\Delta n_{\mathrm{gas}} = 1 - 2 = -1$，两分子气体加氢生成一分子乙烷，系统可及微观状态减少，$\Delta S^\circ = -120\ \mathrm{J/K}$。查表：$S^\circ(\ce{C2H4})=219.4$，$S^\circ(\ce{H2})=130.58$，$S^\circ(\ce{C2H6})=229.5\ \mathrm{J/(mol\cdot K)}$。
> - **(b)** 正确。$\mathrm{N_2O_4}$ 解离为 2 mol $\mathrm{NO_2}$，$\Delta n_{\mathrm{gas}} = +1$，分子数增多、熵增，$\Delta S^\circ = +177\ \mathrm{J/K}$。这是 19.43 中“气体摩尔数增加 → 熵增”的直接应用。
> - **(c)** 正确。固体分解产生 $\ce{H2O(g)}$ 是熵增的主因——气相 $S^\circ$ 远大于固相，即使 $\ce{BeO(s)}$ 的 $S^\circ$ 低于 $\ce{Be(OH)2(s)}$，净效应仍为 $\Delta S^\circ = +152\ \mathrm{J/K}$。与 19.43(b) 同类：“生成气体”往往决定符号。
> - **(d)** 正确。两侧均为气相，$\Delta n_{\mathrm{gas}} = 6 - 5 = +1$，甲醇完全氧化生成更多气体分子，$\Delta S^\circ = +92.3\ \mathrm{J/K}$。数值比 (b)(c) 小，因为虽然分子数增加，但复杂分子（$\ce{CH3OH}$、$\ce{O2}$）变为较简单的 $\ce{CO2}$、$\ce{H2O}$，单摩尔熵的变化部分抵消了摩尔数增加的效果——这说明**不能只看 $\Delta n_{\mathrm{gas}}$**，最终仍需查表求和。
>
> 小结：通用公式 $\Delta S^\circ = \sum n S^\circ(\text{生成物}) - \sum n S^\circ(\text{反应物})$。预判符号时优先看 $\Delta n_{\mathrm{gas}}$ 与是否产生气体；有固体参与或分子复杂度变化较大时，必须以查表计算为准。代码结构清晰，建议保持。
:::

::: {.callout-tip title="习题 19.90"}
反应 $2\mathrm{Mg}(s) + \mathrm{O}_{2}(g) \longrightarrow 2\mathrm{MgO}(s)$ 高度自发。某同学计算该反应的熵变，得到 $\Delta S^{\circ}$ 为很大的负值。该同学在计算中是否出错？请解释。[第 19.4 节]

**解：**

In [69]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

# Mg(s)
S_1 = Q_(32.51, ureg.J / ureg.mol / ureg.K)
n_1 = Q_(2, ureg.mol)
# O2(g)
S_2 = Q_(205.0, ureg.J / ureg.mol / ureg.K)
n_2 = Q_(1, ureg.mol)
# MgO(s)
S_3 = Q_(26.8, ureg.J / ureg.mol / ureg.K)
n_3 = Q_(2, ureg.mol)

Delta_S = (S_3 * n_3) - (S_1 * n_1 + S_2 * n_2)
Delta_S = Delta_S.to(ureg.J / ureg.K)
display(Markdown(fr"$\Delta S^{{\circ}}$ = {Delta_S.magnitude:.3g} J/K"))

$\Delta S^{\circ}$ = -216 J/K

通过 python 计算，该同学的计算中没有出错，$\Delta S^{\circ}$ 确实为很大的负值。

考虑到这个反应高度自发，那么我们还需要分析它的焓变值：

In [70]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

# Mg(s), ΔH_f° = 0 kJ/mol（附录 C）
H_1 = Q_(0, ureg.kJ / ureg.mol)
n_1 = Q_(2, ureg.mol)
# O2(g), ΔH_f° = 0 kJ/mol（附录 C）
H_2 = Q_(0, ureg.kJ / ureg.mol)
n_2 = Q_(1, ureg.mol)
# MgO(s), ΔH_f° = -601.8 kJ/mol（附录 C）
H_3 = Q_(-601.8, ureg.kJ / ureg.mol)
n_3 = Q_(2, ureg.mol)

Delta_H = (H_3 * n_3) - (H_1 * n_1 + H_2 * n_2)
Delta_H = Delta_H.to(ureg.kJ)
display(Markdown(fr"$\Delta H^{{\circ}}$ = {Delta_H.magnitude:.3g} kJ"))

$\Delta H^{\circ}$ = -1.2e+03 kJ

可以看到，这个反应的焓变的绝对值远远大于熵变的绝对值。该习题出现在第 4 节，所以我们不用吉布斯自由能的概念。我们在这里能够得出的结论是，如果一个反应的放热焓变绝对值远远大于无序熵变的绝对值，那么这个反应的自发性由焓变符号决定。（但是活化能呢，这样没有考虑到动力学知识。我也搞不明白了，等老师点评。）

---

> **点评人： Composer 2.5**
>
> 整体评价：**前半段非常好**——你正确判断同学**没有算错**（$\Delta S^\circ_{\mathrm{sys}} = -216\ \mathrm{J/K}$），并主动算了 $\Delta H^\circ \approx -1204\ \mathrm{kJ}$，说明已感到「系统熵减却自发」的矛盾。卡壳点在于：第 4 节后半 precisely 就是为这种矛盾准备的——**不必等吉布斯自由能**，用「环境的熵变」就能串起来。
>
> **与「环境的熵变」怎么关联？** 关键公式（恒压、298 K）：
>
> $$\Delta S_{\mathrm{surr}} = \frac{-\Delta H_{\mathrm{sys}}}{T}$$
>
> 以**反应物/生成物**为系统时，该反应**强烈放热**（$\Delta H^\circ < 0$），系统向环境释放热量，故 $q_{\mathrm{sys}} < 0$。环境吸热，熵**增加**：
>
> $$\Delta S_{\mathrm{surr}} = \frac{-(-1204\times 10^{3}\ \mathrm{J})}{298\ \mathrm{K}} \approx +4.0\times 10^{3}\ \mathrm{J/K}$$
>
> 宇宙总熵变：
>
> $$\Delta S_{\mathrm{univ}} = \Delta S_{\mathrm{sys}} + \Delta S_{\mathrm{surr}} \approx -216 + 4040 \approx +3800\ \mathrm{J/K} > 0$$
>
> 因此反应自发——**不是因为「系统熵增」**，而是**环境熵增的幅度远大于系统熵减**。这与 19.4 节合成氨例题（$\Delta S^\circ_{\mathrm{sys}} = -198\ \mathrm{J/K}$，$\Delta S^\circ_{\mathrm{surr}} \approx +310\ \mathrm{J/K}$，$\Delta S^\circ_{\mathrm{univ}} > 0$）完全同一逻辑，只是镁燃烧放热更猛，环境熵增更大。
>
> **对你现有结论的修正：**
> - 「自发性由焓变符号决定」——**不够准确**。更准确的说法是：在 $\Delta S_{\mathrm{sys}} < 0$ 时，**只有足够大的放热**（使 $\Delta S_{\mathrm{surr}} > |\Delta S_{\mathrm{sys}}|$）才能让 $\Delta S_{\mathrm{univ}} > 0$。若换成一个弱放热且 $\Delta S_{\mathrm{sys}} < 0$ 的反应，仍可能非自发。
> - 「不用吉布斯自由能」——**对，本题用 19.4 的工具就够了**；$\Delta G = \Delta H - T\Delta S$ 只是把上述关系打包，并非唯一途径。
> - 「活化能 / 动力学」——你的直觉很好：热力学说**能自发**，动力学说**有多快**；镁带在常温空气中不会瞬间烧尽，是因为有活化能屏障，与 $\Delta S_{\mathrm{univ}}$ 判据不矛盾。
>
> **一句话总结：** 同学算的是**系统**标准熵变，没有错；「高度自发」要看**宇宙**熵变——系统熵减被环境的熵增抵消并超过，这正是第 4 节「环境的熵变」的核心。
:::

## 第 5 节 吉布斯自由能

反应自发性同时涉及焓与熵：吸热过程可因系统熵增而自发，放热过程也可在系统熵减时自发（见第 4 节环境的熵变）。J. Willard Gibbs 将二者合并为**吉布斯自由能**（自由能）$G$：

$$
G = H - TS
$$

等温过程（尤其恒 $T$、$P$ 下的化学反应）：

$$
\Delta G = \Delta H - T\Delta S
$$

标准条件下：

$$
\Delta G^{\circ} = \Delta H^{\circ} - T\Delta S^{\circ}
$$

由 $\Delta S_{\mathrm{univ}} = \Delta S_{\mathrm{sys}} - \Delta H_{\mathrm{sys}}/T$ 可得 $\Delta G = -T\Delta S_{\mathrm{univ}}$（恒 $T$、$P$）。因此**只需系统的 $\Delta G$**，不必单独算环境熵变，即可判断自发性：

1. $\Delta G < 0$：正向自发；
2. $\Delta G = 0$：平衡；
3. $\Delta G > 0$：正向非自发，逆向自发。

自发过程中 $G$ 不断降低直至极小值（平衡），可类比为「自由能下坡」（`习题 19.8` 附图）。非平衡时：$Q < K$ 则正向自发，$Q > K$ 则逆向自发。

已知 $\Delta H^{\circ}$、$\Delta S^{\circ}$ 和 $T$ 时，用 $\Delta G^{\circ} = \Delta H^{\circ} - T\Delta S^{\circ}$ 计算并判自发——`习题 19.57`。**注意单位**：$\Delta H^{\circ}$ 常用 kJ，$\Delta S^{\circ}$ 常用 J/K，相乘前须统一（如将 $T\Delta S^{\circ}$ 换算为 kJ）。

### 标准生成自由能

**标准生成自由能** $\Delta G_f^{\circ}$：由标准态元素生成 1 mol 该物质时的 $\Delta G^{\circ}$；标准态元素 $\Delta G_f^{\circ} = 0$。298 K 数据见附录 C。反应的标准自由能变：

$$
\Delta G^{\circ} = \sum n \Delta G_f^{\circ}(\text{products}) - \sum m \Delta G_f^{\circ}(\text{reactants})
$$

与 $\Delta H^{\circ}$、$\Delta S^{\circ}$ 的求和形式相同。查表时须注意**相态**（如 $\mathrm{H_2O(l)}$ 与 $\mathrm{H_2O(g)}$ 不同）。逆反应的 $\Delta G^{\circ}$ 变号。

$\Delta H^{\circ}$ 与 $\Delta S^{\circ}$ 同号或异号时，反应自发性随温度变化不同——`习题 19.65` 的表 19.3 四类型即对此的系统总结：(i) $\Delta H<0,\ \Delta S>0$，全温度自发；(ii) $\Delta H>0,\ \Delta S<0$，永不自发；(iii) 两负，低温自发；(iv) 两正，高温自发。

恒 $T$、$P$ 下，$\Delta G$ 还等于系统对环境可做的**最大有用功**：$\Delta G = -w_{\max}$（系统对外做功时 $w < 0$）。热力学判自发，动力学决定速率；$\Delta G^{\circ} < 0$ 不保证反应快速发生。

::: {.callout-tip title="习题 19.8"}
附图显示了某假想反应的 $\Delta G$ 随温度如何变化。（a）在什么温度下系统处于平衡？（b）在哪些温度范围内反应是自发的？（c）$\Delta H$ 为正还是为负？（d）$\Delta S$ 为正还是为负？[第 19.5 节和第 19.6 节]

![](./assets/images/5e2dc6cc9e199162ef75324e18ca23597efd065acd2632f2d513e8e59bafae2a.jpg){fig-align="center"}

温度

**解：**

**(a)** 温度为 250 K 时，$\Delta G = 0$，此时反应处于平衡状态。 

**(b)** $\Delta G \lt 0$ 时，反应为自发的，此时温度 $T \gt 250 K$。

**(c)** $\Delta H \gt 0$，可以想象温度为 0 K，沿横轴一直向左观察，发现 $\Delta H$ 最终为正值。

**(d)** $\Delta S$ 为正。$-T \Delta S$ 为负，$T \Delta S$ 为正，因 $T \gt 0$，故 $\Delta S \gt 0$。

---

> **点评人： Composer 2.5**
>
> 整体评价：**四问全部正确！** 你已能从 $\Delta G$–$T$ 图读出平衡温度、自发区间，并反推 $\Delta H$、$\Delta S$ 的符号，与第 19.5–19.6 节的核心公式 $\Delta G = \Delta H - T\Delta S$ 完全一致。
>
> - **(a)** 正确。附图红线与 $\Delta G = 0$ 虚线交于 **250 K**，此处 $\Delta G = 0$，系统处于平衡。
> - **(b)** 正确。$\Delta G < 0$ 时反应自发；红线在 250 K 以**右**位于零线下方，故 **$T > 250\ \mathrm{K}$** 自发；$T < 250\ \mathrm{K}$ 时 $\Delta G > 0$，非自发。
> - **(c)** 正确。取 $T \to 0\ \mathrm{K}$ 时 $\Delta G \approx \Delta H$（$T\Delta S$ 项可忽略），红线在零线上方，故 **$\Delta H > 0$**（吸热反应）。你「沿横轴向左看」的直觉对，表述上可更精确地写：读 **$T = 0$ 处的纵截距**即为 $\Delta H$。
> - **(d)** 结论正确。更简洁的读图法：$\Delta G$–$T$ 为直线，**斜率** $= -\Delta S$；红线向下倾斜，斜率为负，故 **$\Delta S > 0$**。你的论证（高温时 $\Delta G < 0$ 且已知 $\Delta H > 0$，必要求 $T\Delta S > \Delta H$，故 $\Delta S > 0$）也成立，只是 (d) 单独作答时直接用斜率最快。
>
> 小结：该图对应表 19.3 的 **类型 (iv)**——**$\Delta H > 0$、$\Delta S > 0$**，低温非自发、高温自发，平衡温度 $T = \Delta H / \Delta S = 250\ \mathrm{K}$。
:::


::: {.callout-tip title="习题 19.57"}
某化学反应的 $\Delta H^{\circ} = -35.4$ kJ，$\Delta S^{\circ} = -85.5$ J/K。（a）该反应是放热还是吸热？（b）该反应使系统的无序度增加还是减小？（c）计算 298 K 时该反应的 $\Delta G^{\circ}$。（d）在 298 K 标准条件下该反应是否自发？

**解：**

**(a)** 该反应是放热的，因为 $\Delta H^{\circ} \lt 0$。

**(b)** 该反应使系统的无序度减小，因为 $\Delta S^{\circ} \lt 0$。

**(c)**

In [71]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

H = Q_(-35.4, ureg.kJ)
S = Q_(-85.5, ureg.J / ureg.K)
T = Q_(298.15, ureg.K)

G = H - T * S
G = G.to(ureg.kJ)
display(Markdown(fr"$\Delta G^{{\circ}}$ = {G.magnitude:.3g} kJ/mol"))

$\Delta G^{\circ}$ = -9.91 kJ/mol

**(d)** 自发。因为 $\Delta G^{\circ} \lt 0$。

---

> **点评人： Composer 2.5**
>
> 整体评价：**四问全部正确。** (a)(b) 符号判断简洁准确；(c) 代码用 $\Delta G^{\circ} = \Delta H^{\circ} - T\Delta S^{\circ}$ 得到 **$-9.91\ \mathrm{kJ/mol}$**（取 $T = 298.15\ \mathrm{K}$ 时；若用 $298\ \mathrm{K}$ 约为 $-9.92\ \mathrm{kJ}$，差别可忽略）；(d) 因 $\Delta G^{\circ} < 0$，298 K 标准条件下**自发**，正确。
>
> **关于单位要不要写 `/mol`？** 你的困惑很正常，分两层理解：
>
> 1. **附录 C** 中的 $\Delta H_f^{\circ}$、$S^{\circ}$ 是**单种物质**的量，单位明确为 $\mathrm{kJ/mol}$、$\mathrm{J/(mol\cdot K)}$（每摩尔该物质）。
> 2. **本题及 19.65 等** 给出的 $\Delta H^{\circ}$、$\Delta S^{\circ}$、$\Delta G^{\circ}$ 是**整个配平反应**的标准量，按「**1 mol 反应进度**」定义——即化学方程式中各物种系数所对应的那一份反应发生时的焓变/熵变/自由能变。严格写应是：
>    - $\Delta H^{\circ}$：$\mathrm{kJ/mol}$（每 mol 反应）
>    - $\Delta S^{\circ}$：$\mathrm{J/(mol\cdot K)}$
>    - $\Delta G^{\circ}$：$\mathrm{kJ/mol}$
>    教材为简洁常省略 `/mol`，只写 $\mathrm{kJ}$、$\mathrm{J/K}$；**数值与含 `/mol` 时相同**，因为这里的 mol 指「反应进度」，不是某一种反应物。
>
> **实用建议：** 题目未写 `/mol` 时，答案写 **$-9.91\ \mathrm{kJ}$** 或 **$-9.91\ \mathrm{kJ/mol}$** 均可；你代码输出 `kJ/mol` 更规范，与附录 C 习惯一致，**不必改**。计算时只要 $\Delta H^{\circ}$ 与 $T\Delta S^{\circ}$ 用同一份「每 mol 反应」的量即可（你的 pint 写法自洽）。
>
> 验算：$\Delta G^{\circ} = -35.4\ \mathrm{kJ} - (298\ \mathrm{K})(-85.5\ \mathrm{J/K}) = -35.4 + 25.5\ \mathrm{kJ} = -9.9\ \mathrm{kJ}$。
:::

::: {.callout-tip title="习题 19.65"}
将下列各反应归类为表 19.3 总结的四种类型之一：（i）所有温度下均自发；（ii）任何温度下均不自发；（iii）低温自发、高温不自发；（iv）高温自发、低温不自发。

(a) $\mathrm{N}_{2}(g) + 3\mathrm{H}_{2}(g) \longrightarrow 2\mathrm{NH}_{3}(g)$; $\Delta H^{\circ} = -91.8\ \mathrm{kJ}$; $\Delta S^{\circ} = -197\ \mathrm{J/K}$

(b) $\mathrm{S}_{8}(s) + 12\mathrm{O}_{2}(g) \longrightarrow 8\mathrm{SO}_{3}(g)$; $\Delta H^{\circ} = -3168\ \mathrm{kJ}$; $\Delta S^{\circ} = 2053.6\ \mathrm{J/K}$

(c) $2\mathrm{POCl}_{3}(g) \longrightarrow 2\mathrm{PCl}_{3}(g) + \mathrm{O}_{2}(g)$; $\Delta H^{\circ} = +572\ \mathrm{kJ}$; $\Delta S^{\circ} = +179\ \mathrm{J/K}$

**解：**

**(a)** 低温自发、高温不自发。

**(b)** 所有温度下均自发。

**(c)** 高温自发、低温不自发。

---

> **点评人： Composer 2.5**
>
> 整体评价：**三题分类全部正确！** 你已掌握用 $\Delta H^{\circ}$、$\Delta S^{\circ}$ 的符号组合判断反应与温度的关系，这正是表 19.3 的核心。
>
> | 小题 | $\Delta H^{\circ}$ | $\Delta S^{\circ}$ | 你的归类 | 表 19.3 |
> |------|-------------------|-------------------|---------|---------|
> | (a) 合成氨 | $-$ | $-$ | 低温自发、高温不自发 | **(iii)** |
> | (b) 硫燃烧 | $-$ | $+$ | 所有温度均自发 | **(i)** |
> | (c) POCl₃ 分解 | $+$ | $+$ | 高温自发、低温不自发 | **(iv)** |
>
> - **(a)** 正确。**$\Delta H^{\circ} < 0$、$\Delta S^{\circ} < 0$** → 类型 **(iii)**。低温时 $|\Delta H^{\circ}| > |T\Delta S^{\circ}|$，焓驱动自发；高温时熵项占主导，$\Delta G^{\circ} > 0$ 非自发。平衡温度 $T \approx \Delta H^{\circ}/\Delta S^{\circ} = 91800/197 \approx 466\ \mathrm{K}$。
> - **(b)** 正确。**$\Delta H^{\circ} < 0$、$\Delta S^{\circ} > 0$** → 类型 **(i)**。$\Delta G^{\circ} = \Delta H^{\circ} - T\Delta S^{\circ}$ 两项均使 $\Delta G^{\circ} < 0$，故**任何温度**下均自发。
> - **(c)** 正确。**$\Delta H^{\circ} > 0$、$\Delta S^{\circ} > 0$** → 类型 **(iv)**，与 `习题 19.8` 附图同类。低温时吸热项占主导，非自发；高温时 $T\Delta S^{\circ} > \Delta H^{\circ}$，自发。
>
> 小建议：题目要求对应表 19.3 的 **(i)–(iv)** 编号，作答时可写「**(iii) 低温自发、高温不自发**」，中英文描述与编号并列，更贴题意。记忆口诀：$\Delta H$、$\Delta S$ **同号** → (iii) 或 (iv)（看谁为负）；**异号**且 $\Delta H < 0$、$\Delta S > 0$ → (i)；**异号**且 $\Delta H > 0$、$\Delta S < 0$ → (ii)（本题未出现）。
:::


## 第 6 节 自由能与温度

附录 C 的 $\Delta G_f^{\circ}$ 只对应 298 K，而反应常在其他温度进行。把 $\Delta G$ 拆成**焓项**与**熵项**便能看清温度的作用：

$$
\Delta G = \underbrace{\Delta H}_{\text{焓项}} + \underbrace{(-T\Delta S)}_{\text{熵项}}
$$

- **焓项 $\Delta H$**：可正可负，随温度变化很小。
- **熵项 $-T\Delta S$**：符号由 $\Delta S$ 决定（$\Delta S>0 \Rightarrow -T\Delta S<0$），其**绝对值随 $T$ 升高而增大**。因此温度越高，熵项在决定 $\Delta G$ 符号时越占主导。

### 两项符号相反时，温度决定自发性

当 $\Delta H$ 与 $-T\Delta S$ **同号**时，$\Delta G$ 的符号与温度无关（全温度自发或全温度不自发）；只有当二者**异号**时，$\Delta G$ 的符号才取决于两项大小，温度才起关键作用——这正是 `习题 19.65` 表 19.3 中 (iii)、(iv) 两类的来源：

- 类型 (iii) $\Delta H<0,\ \Delta S<0$：**低温自发、高温不自发**（如水结冰 $\mathrm{H_2O}(l)\to\mathrm{H_2O}(s)$）。
- 类型 (iv) $\Delta H>0,\ \Delta S>0$：**高温自发、低温不自发**（如冰融化 $\mathrm{H_2O}(s)\to\mathrm{H_2O}(l)$）。

以冰融化为例：$\Delta H>0$、$\Delta S>0$。低于 $0\ ^{\circ}\mathrm{C}$ 时 $|\Delta H|>|-T\Delta S|$，焓项占主导，$\Delta G>0$ 不自发（此时结冰自发）；高于 $0\ ^{\circ}\mathrm{C}$ 时熵项占主导，$\Delta G<0$ 自发；在正常熔点 $0\ ^{\circ}\mathrm{C}$ 两项恰好抵消，$\Delta G=0$，两相平衡。

### 估算其他温度下的 $\Delta G$

若近似认为 $\Delta H^{\circ}$、$\Delta S^{\circ}$ **不随温度变化**，可用 298 K 的数据估算任意温度下的标准自由能变：

$$
\Delta G^{\circ}(T) \approx \Delta H^{\circ} - T\Delta S^{\circ}
$$

这一近似是 `习题 19.69`、`习题 19.83(b)` 的关键——**不能直接把 298 K 的 $\Delta G^{\circ}$ 搬到其他温度**，须用当前温度重算熵项。令 $\Delta G^{\circ}=0$ 还可求得自发性反转的**临界温度** $T = \Delta H^{\circ}/\Delta S^{\circ}$（即相变点或平衡温度）。


::: {.callout-tip title="习题 19.69"}
某反应的 $\Delta H = -32$ kJ，$\Delta S = -98$ J/K。假设 $\Delta H$ 和 $\Delta S$ 不随温度变化。（a）在什么温度下该反应的 $\Delta G = 0$？（b）若温度高于（a）中的值，反应是自发的还是非自发的？

**解：** 

**(a)** 解方程：$0 = (-32000) - T * (-98)$


In [72]:
T = 32e3 / 98
print(f"T = {T:.3g} K")

T = 327 K


**(b)** 高于 (a) 中的值时，反应为非自发。

---

> **点评人： Composer 2.5**
>
> 整体评价：**两问均正确。** 你已会用 $\Delta G = \Delta H - T\Delta S = 0$ 求平衡温度，并判断高温下的自发性；与 `习题 19.65` 类型 **(iii)**（$\Delta H < 0$、$\Delta S < 0$，低温自发、高温非自发）一致。
>
> - **(a)** 正确。令 $\Delta G = 0$：$\Delta H = T\Delta S$，故
>   $$T = \frac{\Delta H}{\Delta S} = \frac{-32\times 10^{3}\ \mathrm{J}}{-98\ \mathrm{J/K}} \approx 327\ \mathrm{K}$$
>   你先将 $-32\ \mathrm{kJ}$ 换算为 $-32000\ \mathrm{J}$ 再求解，单位处理正确；代码 `32e3/98` 得到 **327 K**（更精确为 $326.5\ \mathrm{K}$，约 **53 °C**）。书写时建议显式写出 $\Delta G = \Delta H - T\Delta S = 0$，比直接列数字方程更规范。
> - **(b)** 正确。$T > 327\ \mathrm{K}$ 时 $|T\Delta S| > |\Delta H|$，熵项 $-T\Delta S$ 为**正**且超过负的 $\Delta H$，故 $\Delta G > 0$，**非自发**。$T < 327\ \mathrm{K}$ 时则 $\Delta G < 0$，自发。
>
> 小结：两负型反应的平衡温度 $T_{\mathrm{eq}} = \Delta H/\Delta S$；本题 $\Delta G$–$T$ 图是一条斜率为 $-\Delta S > 0$ 的直线（与 `习题 19.8` 相反），在 327 K 以左 $\Delta G < 0$，以右 $\Delta G > 0$。
:::

## 第 7 节 自由能与平衡常数

本节把自由能用于两件事：**在非标准条件下求 $\Delta G$**，以及**把 $\Delta G^{\circ}$ 与平衡常数 $K$ 联系起来**。

### 非标准条件下的自由能

标准态（气体 1 atm、溶质 1 M）下才用 $\Delta G^{\circ}$；实际反应多在非标准条件进行。任意条件下的自由能变为：

$$
\Delta G = \Delta G^{\circ} + RT\ln Q
$$

其中 $R = 8.314\ \mathrm{J/(mol\cdot K)}$，$Q$ 为反应商（算法同 $K$，但代入某时刻的浓度/分压）。标准条件下所有浓度为 1，$Q=1$、$\ln Q=0$，退化为 $\Delta G=\Delta G^{\circ}$。计算 $Q$ 时**气体用 atm 分压、溶质用 mol/L**，纯固、纯液、溶剂不计入。这正是 `习题 19.9`、`习题 19.77` 用到的关系：$Q$ 相对 $K$ 越小，$\Delta G$ 越负、正向推动力越大。

### $\Delta G^{\circ}$ 与 $K$ 的关系

平衡时 $\Delta G=0$、$Q=K$，代入上式即得本章最重要的桥梁公式：

$$
\Delta G^{\circ} = -RT\ln K
\qquad\Longleftrightarrow\qquad
K = e^{-\Delta G^{\circ}/RT}
$$

它把 $K$ 与反应的焓变、熵变联系起来。符号对应关系一目了然：

| $\Delta G^{\circ}$ | $\ln K$ | $K$ | 平衡位置 |
| --- | --- | --- | --- |
| $<0$ | $>0$ | $K>1$ | 偏向产物；$\Delta G^{\circ}$ 越负 $K$ 越大 |
| $=0$ | $0$ | $K=1$ | 反应物与产物相当 |
| $>0$ | $<0$ | $K<1$ | 偏向反应物 |

应用要点（见 `习题 19.81`、`习题 19.83`）：

- **单位**：$\Delta G^{\circ}$ 常以 kJ/mol 给出，代入前须换算为 J/mol 以配合 $R$。
- **相态**：气相反应对应 $K_p$、溶液反应对应 $K_c$；异相反应中固、液不出现在 $K$ 中（如 $\mathrm{BaCO_3}$ 分解的 $K_p=P_{\mathrm{CO_2}}$）。
- **温度**：$K$ 随温度变化，因为 $\Delta G^{\circ}(T)\approx\Delta H^{\circ}-T\Delta S^{\circ}$ 依赖温度（承接第 6 节）。

### 反应耦合驱动非自发反应

$\Delta G^{\circ}$ 很正的反应可与一个 $\Delta G^{\circ}$ 很负的自发反应**耦合**，使总反应的 $\Delta G^{\circ}$（两式相加）变负而得以进行。例如从辉铜矿提铜的 $\mathrm{Cu_2S}(s)\to 2\mathrm{Cu}(s)+\mathrm{S}(s)$（$+86.2\ \mathrm{kJ}$）与 $\mathrm{S}(s)+\mathrm{O_2}(g)\to\mathrm{SO_2}(g)$（$-300.4\ \mathrm{kJ}$）耦合，总 $\Delta G^{\circ}=-214.2\ \mathrm{kJ}$。生命体系同理：以葡萄糖氧化释放的自由能经 ATP↔ADP 循环储存、转运，用以驱动体内非自发的合成反应。


::: {.callout-tip title="习题 19.9"}
考虑反应 $\mathrm{A}_{2}(g) + \mathrm{B}_{2}(g) \rightleftharpoons 2\mathrm{AB}(g)$，图中 A 原子用红色表示，B 原子用蓝色表示。（a）若 $K_{c} = 1$，哪个方框代表平衡时的系统？（b）若 $K_{c} = 1$，哪个方框代表 $Q < K_{c}$ 时的系统？（c）按反应 $\Delta G$ 的大小递增排列这些方框。[第 19.5 节和第 19.7 节]

![](./assets/images/807399bf1e1012fbcc660801eed1029411d972477881aa4d9a843eb2d6e45240.jpg){fig-align="center"}

(1)

![](./assets/images/876947b3776cf9cc3d574b1bced4a1eaaebcfb1b58646ed4d4ebd04d04df1ab1.jpg){fig-align="center"}

(2)

![](./assets/images/edc86bf2a89982e74e2492926859cf746ead5af9a8bd641619cb464c7a7bc570.jpg){fig-align="center"}

(3)

**解：**

$$K_c = \frac{[AB]^2}{[A_2][B_2]}$$

图 1：$[A_2]=3,[B_2]=3,[AB]=3$；$Q = \frac{3^2}{3\times 3} = 1$

图 2：$[A_2]=4,[B_2]=4,[AB]=1$；$Q = \frac{1^2}{4\times 4} = \frac{1}{16}$

图 3：$[A_2]=1,[B_2]=1,[AB]=7$；$Q = \frac{7^2}{1\times 1} = 49$

**(a)** 图 1 代表平衡时的系统，因为此时 $Q = K_c = 1$。

**(b)** 图 2，此时 $Q = \frac{1}{16} \lt K_c$。

**(c)** 三幅图中的 $\Delta G^{\circ}$ 均相同，只需要考虑 $+ R T \ln Q$ 的大小。$R T$ 为常数，故只要比较 $\ln Q$ 即 $Q$ 的大小。那么显然按 $\Delta G$ 的大小递增排序为 2 -> 1 -> 3。

---

> **点评人： Composer 2.5**
>
> 整体评价：三问全部正确。你对三幅图逐一数分子、写 $Q_c$ 并和 $K_c=1$ 对照，步骤清楚；(c) 用 $\Delta G=\Delta G^\circ+RT\ln Q$ 在同一温度下比较三框，逻辑也对。
>
> - **读图与 $Q_c$**：图 1 为 3 个 $\mathrm{A_2}$、3 个 $\mathrm{B_2}$、3 个 $\mathrm{AB}$；图 2 为 4、4、1；图 3 为 1、1、7。三框体积相同，分子数可直接当作相对“浓度”代入 $Q_c=[\mathrm{AB}]^2/([\mathrm{A_2}][\mathrm{B_2}])$，得 $Q_1=1$、$Q_2=1/16$、$Q_3=49$，与配图一致。
> - **(a)** 正确。$Q_1=K_c=1$，图 1 为平衡态。
> - **(b)** 正确。$Q_2=1/16<K_c$，图 2 产物偏少、反应物偏多，系统将向生成 $\mathrm{AB}$ 方向进行直至 $Q$ 趋近 $K$。
> - **(c)** 正确。三框同温同反应，$\Delta G^\circ$ 相同，故 $\Delta G$ 的大小由 $RT\ln Q$ 决定；$Q_2<Q_1<Q_3$，故 $\Delta G$ 递增为 **2 → 1 → 3**。因 $K_c=1$ 时 $\Delta G^\circ=-RT\ln K=0$，还可直接写 $\Delta G=RT\ln Q$：图 2 为负（正向自发）、图 1 为零（平衡）、图 3 为正（正向非自发）。小建议：(c) 中比较的是各**非标准态**下的 $\Delta G$，不是 $\Delta G^\circ$；写“$\Delta G^\circ$ 相同，只需比较 $RT\ln Q$”更严谨。
>
> 小结：$Q<K$ 时反应正向进行、$Q>K$ 时逆向进行、$Q=K$ 时平衡；排序题在同一 $T$ 下常化为比较 $Q$ 或 $\ln Q$。
:::



::: {.callout-tip title="习题 19.77"}
当 $O_{2}$ 分压减小时，指出下列各反应中 $\Delta G$ 增加、减小还是不变：

(a) $2\mathrm{CO}(g) + \mathrm{O}_{2}(g) \longrightarrow 2\mathrm{CO}_{2}(g)$

(b) $2\mathrm{H}_{2}\mathrm{O}_{2}(l)\longrightarrow2\mathrm{H}_{2}\mathrm{O}(l)+\mathrm{O}_{2}(g)$

(c) $2\mathrm{KClO}_{3}(s)\longrightarrow\mathrm{KCl}(s)+3\mathrm{O}_{2}(g)$

**解：**

**(a)**

$$
Q = \frac{[CO_2]^2}{[CO][O_2]} = \frac{P_{CO_2}^2}{P_{CO}P_{O_2}}
$$

$O_{2}$ 位于 $Q$ 的分母，当 $O_{2}$ 分压减小时，$Q$ 增加。根据 $\Delta G = \Delta G^{\circ} + R T \ln Q$ 可知，$\Delta G$ 增加。

**(b)** $\Delta G$ 减小。思路和 (a) 相同，不过此时 $O_{2}$ 位于 $Q$ 的分子。

**(c)** $\Delta G$ 减小。同 (b)。

我的补充：平衡常数中，如果出现了气相物质，那么其它相的物质可以不考虑。气相物质在平衡常数的计算中占主导地位。

---

> **点评人： Composer 2.5**
>
> 整体评价：三问判断全部正确。你用 $\Delta G=\Delta G^\circ+RT\ln Q$ 把「$O_2$ 分压变化 → $Q$ 变化 → $\Delta G$ 变化」串起来，是这类定性题的标准思路；(a) 写 $Q$ 表达式最完整，(b)(c) 结论对，若也写出 $Q$ 会更对称。
>
> - **(a)** 正确。$Q=P_{\mathrm{CO_2}}^2/(P_{\mathrm{CO}}P_{\mathrm{O_2}})$，$O_2$ 在分母；$P_{\mathrm{O_2}}$ 减小 → $Q$ 增大 → $\ln Q$ 增大 → $\Delta G$ **增加**。物理上：生成物侧相对富集，系统更「偏离」正向平衡，正向进行的驱动力减弱（若原先处于平衡附近，则平衡将左移）。
> - **(b)** 正确。纯液体不出现在 $Q$ 中，$Q=P_{\mathrm{O_2}}$；$P_{\mathrm{O_2}}$ 减小 → $Q$ 减小 → $\Delta G$ **减小**。
> - **(c)** 正确。固体 $\mathrm{KClO_3}$、$\mathrm{KCl}$ 活度取 1，$Q=P_{\mathrm{O_2}}^3$；$P_{\mathrm{O_2}}$ 减小 → $Q$ 显著减小 → $\Delta G$ **减小**。注意 $O_2$ 在 (c) 中指数为 3，$P_{\mathrm{O_2}}$ 小幅下降时 $Q$ 的变化比 (b) 更敏感，但「减小 → $\Delta G$ 减小」的方向不变。
>
> 关于你的补充：异相平衡中，**纯固体、纯液体**（或其标准态溶液）活度 ≈ 1，故不出现在 $Q$、$K$ 中；表达式里通常只保留**气体分压**（或溶液中物种的活度/浓度）。更准确的说法是「非气相纯相可省略」，而不是「有气相时其它相可忽略」——例如 $K$ 中仍隐含各固、液相处于标准态。
>
> 小结：先写 $Q$，看 $O_2$ 在分子还是分母、指数多少；在其它分压/浓度不变的假设下，$P_{\mathrm{O_2}}\downarrow$ 使 $Q\downarrow$（$O_2$ 在分子）或 $Q\uparrow$（$O_2$ 在分母），再据 $\Delta G=\Delta G^\circ+RT\ln Q$ 判断 $\Delta G$ 增减。
:::



::: {.callout-tip title="习题 19.81"}
利用附录 C 中的数据，计算 298 K 时下列各反应的平衡常数 $K$ 和 $\Delta G^{\circ}$：

(a) $\mathrm{H}_{2}(g) + \mathrm{I}_{2}(g) \rightleftharpoons 2 \mathrm{HI}(g)$

(b) $\mathrm{C_{2}H_{5}OH(g)} \rightleftharpoons \mathrm{C_{2}H_{4}(g)} + \mathrm{H_{2}O(g)}$

(c) $3\mathrm{C}_{2}\mathrm{H}_{2}(g) \rightleftharpoons \mathrm{C}_{6}\mathrm{H}_{6}(g)$

**解：** $\Delta G^{\circ}$ 与 $K$ 的关系为：

$$
K = e^{-\Delta G^{\circ}/RT}
$$

解题步骤为：

- 通过附录 C 查询反应式中各物质的 $\Delta G^{\circ}_f$ 值
- 按 $\Delta G^{\circ} = \sum n \Delta G_f^{\circ}(\text{生成物}) - \sum m \Delta G_f^{\circ}(\text{反应物})$ 计算反应式整体的 $\Delta G^{\circ}$
- 代入上述关系式求解 $K$

**(a)** 查附录 C：$\Delta G_f^{\circ}(\mathrm{H_2}) = 0$，$\Delta G_f^{\circ}(\mathrm{I_2,g}) = 19.37\ \mathrm{kJ/mol}$，$\Delta G_f^{\circ}(\mathrm{HI}) = 1.30\ \mathrm{kJ/mol}$。注意反应物为 $\mathrm{I_2(g)}$ 而非 $\mathrm{I_2(s)}$。预测：生成物 $\Delta G_f^{\circ}$ 之和低于反应物，$\Delta G^{\circ} < 0$，故 $K > 1$。



In [73]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

# H2(g), ΔG_f° = 0 kJ/mol（附录 C）
G_1 = Q_(0, ureg.kJ / ureg.mol)
n_1 = Q_(1, ureg.mol)
# I2(g), ΔG_f° = 19.37 kJ/mol（附录 C）
G_2 = Q_(19.37, ureg.kJ / ureg.mol)
n_2 = Q_(1, ureg.mol)
# HI(g), ΔG_f° = 1.30 kJ/mol（附录 C）
G_3 = Q_(1.30, ureg.kJ / ureg.mol)
n_3 = Q_(2, ureg.mol)

Delta_G = (G_3 * n_3) - (G_1 * n_1 + G_2 * n_2)
Delta_G = Delta_G.to(ureg.kJ)

T = Q_(298.15, ureg.K)
R = Q_(8.314, ureg.J / ureg.mol / ureg.K)
K = np.exp(-Delta_G.to(ureg.J).magnitude / (R.magnitude * T.magnitude))

display(Markdown(
    fr"$\Delta G^{{\circ}}$ = {Delta_G.magnitude:.3g} kJ/mol，$K$ = {K:.3g}"
))


$\Delta G^{\circ}$ = -16.8 kJ/mol，$K$ = 867

计算结果符合预期：$\Delta G^{\circ} < 0$，$K \gg 1$，298 K 下 HI 的生成反应在标准态下有利。

**(b)** 查附录 C：$\Delta G_f^{\circ}(\mathrm{C_2H_5OH,g}) = -168.5\ \mathrm{kJ/mol}$，$\Delta G_f^{\circ}(\mathrm{C_2H_4}) = 68.11\ \mathrm{kJ/mol}$，$\Delta G_f^{\circ}(\mathrm{H_2O,g}) = -228.57\ \mathrm{kJ/mol}$。预测：生成物 $\Delta G_f^{\circ}$ 之和高于反应物，$\Delta G^{\circ} > 0$，故 $K < 1$。


In [74]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

# C2H5OH(g), ΔG_f° = -168.5 kJ/mol（附录 C）
G_1 = Q_(-168.5, ureg.kJ / ureg.mol)
n_1 = Q_(1, ureg.mol)
# C2H4(g), ΔG_f° = 68.11 kJ/mol（附录 C）
G_2 = Q_(68.11, ureg.kJ / ureg.mol)
n_2 = Q_(1, ureg.mol)
# H2O(g), ΔG_f° = -228.57 kJ/mol（附录 C）
G_3 = Q_(-228.57, ureg.kJ / ureg.mol)
n_3 = Q_(1, ureg.mol)

Delta_G = (G_2 * n_2 + G_3 * n_3) - G_1 * n_1
Delta_G = Delta_G.to(ureg.kJ)

T = Q_(298.15, ureg.K)
R = Q_(8.314, ureg.J / ureg.mol / ureg.K)
K = np.exp(-Delta_G.to(ureg.J).magnitude / (R.magnitude * T.magnitude))

display(Markdown(
    fr"$\Delta G^{{\circ}}$ = {Delta_G.magnitude:.3g} kJ/mol，$K$ = {K:.3g}"
))


$\Delta G^{\circ}$ = 8.04 kJ/mol，$K$ = 0.039

计算结果符合预期：$\Delta G^{\circ} > 0$，$K < 1$，298 K 下气相乙醇脱水分解在标准态下不利。

**(c)** 查附录 C：$\Delta G_f^{\circ}(\mathrm{C_2H_2}) = 209.2\ \mathrm{kJ/mol}$，$\Delta G_f^{\circ}(\mathrm{C_6H_6,g}) = 129.7\ \mathrm{kJ/mol}$。预测：三分子乙炔聚合成苯，生成物 $\Delta G_f^{\circ}$ 远低于反应物，$\Delta G^{\circ} \ll 0$，$K \gg 1$。


In [75]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

# C2H2(g), ΔG_f° = 209.2 kJ/mol（附录 C）
G_1 = Q_(209.2, ureg.kJ / ureg.mol)
n_1 = Q_(3, ureg.mol)
# C6H6(g), ΔG_f° = 129.7 kJ/mol（附录 C）
G_2 = Q_(129.7, ureg.kJ / ureg.mol)
n_2 = Q_(1, ureg.mol)

Delta_G = (G_2 * n_2) - G_1 * n_1
Delta_G = Delta_G.to(ureg.kJ)

T = Q_(298.15, ureg.K)
R = Q_(8.314, ureg.J / ureg.mol / ureg.K)
K = np.exp(-Delta_G.to(ureg.J).magnitude / (R.magnitude * T.magnitude))

display(Markdown(
    fr"$\Delta G^{{\circ}}$ = {Delta_G.magnitude:.3g} kJ/mol，$K$ = {K:.3g}"
))


$\Delta G^{\circ}$ = -498 kJ/mol，$K$ = 1.71e+87

计算结果符合预期：$\Delta G^{\circ} \ll 0$，$K$ 极大，说明 298 K 标准态下乙炔三聚生成苯在热力学上高度有利（实际速率可能受动力学控制）。
:::

::: {.callout-tip title="习题 19.83"}
考虑碳酸钡的分解：

$$
\mathrm{BaCO}_{3} (s) \rightleftharpoons \mathrm{BaO} (s) + \mathrm{CO} _{2} (g)
$$

利用附录 C 中的数据，计算（a）298 K 和（b）1100 K 时 $CO_{2}$ 的平衡分压。

**解：**

**(a)**

In [76]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

# $\mathrm{BaCO}_{3} (s)$
G_1 = Q_(-1137.6, ureg.kJ / ureg.mol)
n_1 = Q_(1, ureg.mol)
# $\mathrm{BaO} (s)$
G_2 = Q_(-525.1, ureg.kJ / ureg.mol)
n_2 = Q_(1, ureg.mol)
# $\mathrm{CO} _{2} (g)$
G_3 = Q_(-394.4, ureg.kJ / ureg.mol)
n_3 = Q_(1, ureg.mol)

G = G_3 * n_3 + G_2 * n_2 - G_1 * n_1

R = Q_(8.314, ureg.J / ureg.mol / ureg.K)
T = Q_(298.15, ureg.K)

K = np.exp(- G.to(ureg.J).magnitude / R.magnitude / T.magnitude)
K = K

P = K
display(Markdown(rf"$CO_2$ 的平衡分压为 {P:.3g} atm"))

$CO_2$ 的平衡分压为 6.14e-39 atm

**(b)**


In [77]:
#| code-fold: true

import numpy as np
from IPython.display import Markdown
import pint

ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

# $\mathrm{BaCO}_{3} (s)$
G_1 = Q_(-1137.6, ureg.kJ / ureg.mol)
n_1 = Q_(1, ureg.mol)
# $\mathrm{BaO} (s)$
G_2 = Q_(-525.1, ureg.kJ / ureg.mol)
n_2 = Q_(1, ureg.mol)
# $\mathrm{CO} _{2} (g)$
G_3 = Q_(-394.4, ureg.kJ / ureg.mol)
n_3 = Q_(1, ureg.mol)

G = G_3 * n_3 + G_2 * n_2 - G_1 * n_1

R = Q_(8.314, ureg.J / ureg.mol / ureg.K)
T = Q_(1100, ureg.K)

K = np.exp(- G.to(ureg.J).magnitude / R.magnitude / T.magnitude)
K = K

P = K
display(Markdown(rf"$CO_2$ 的平衡分压为 {P:.3g} atm"))

$CO_2$ 的平衡分压为 4.39e-11 atm

---

> **点评人： Composer 2.5**
>
> 整体评价：**(a) 思路与数值基本正确**；**(b) 是这道题最容易「摸不准」的地方**——你把 298 K 算出的 $\Delta G^{\circ}$ 直接代进 1100 K 的 $K=e^{-\Delta G^{\circ}/RT}$，**没有随温度更新 $\Delta G^{\circ}$**，因此 (b) 的数值偏小约 $10^7$ 倍。下面按「这道题在考什么 → 怎么做 → 你哪里对、哪里要改」展开。
>
> ### 1. 这道题的核心：异相平衡 + 自由能与 $K$ 的换算
>
> 反应 $\mathrm{BaCO_3(s) \rightleftharpoons BaO(s) + CO_2(g)}$ 中，两固体处于纯相，活度 $a \approx 1$，**不出现在 $Q$、$K$ 中**。因此（以 $K_p$ 计）：
>
> $$K_p = P_{\mathrm{CO_2}} \quad (\text{单位 atm 时})$$
>
> 这也是你代码里写 `P = K` 的依据——**这一步是对的**，和 `习题 19.77` 中「有气相时固体可省略」是同一类规则。
>
> 计算路线：
>
> 1. 用附录 C 的 $\Delta G_f^{\circ}$（298 K）求 $\Delta G^{\circ}$；
> 2. $K = e^{-\Delta G^{\circ}/RT}$；
> 3. $P_{\mathrm{CO_2}} = K_p$。
>
> ### 2. (a) 298 K：你的做法正确
>
> 查附录 C（$\mathrm{kJ/mol}$）：
>
> | 物种 | $\Delta G_f^{\circ}$ |
> |------|---------------------|
> | $\mathrm{BaCO_3(s)}$ | $-1137.6$ |
> | $\mathrm{BaO(s)}$ | $-525.1$ |
> | $\mathrm{CO_2(g)}$ | $-394.4$ |
>
> $$\Delta G^{\circ} = (-525.1) + (-394.4) - (-1137.6) = +218.1\ \mathrm{kJ/mol}$$
>
> $\Delta G^{\circ} \gg 0$ → 298 K 标准态下**分解非自发**，$K_p$ 极小，与你算出的 $P_{\mathrm{CO_2}} \approx 6.1\times10^{-39}\ \mathrm{atm}$ 一致。物理解释：**室温下碳酸钡极稳定**，平衡时 $\mathrm{CO_2}$ 分压小到可以视为零，完全合理。
>
> 手算验算：$K = \exp\!\bigl(-218100/(8.314\times298.15)\bigr) \approx 6\times10^{-39}$。
>
> ### 3. (b) 1100 K：不能只用 298 K 的 $\Delta G^{\circ}$
>
> **常见误区**：在 (a) 里算好 $\Delta G^{\circ}=+218\ \mathrm{kJ/mol}$ 后，只把 $T$ 改成 1100 再算 $e^{-\Delta G^{\circ}/RT}$。这隐含假设「$\Delta G^{\circ}$ 不随温度变」，对**本反应不成立**。
>
> 从附录 C 还可得（298 K）：
>
> $$\Delta H^{\circ} = (-553.5)+(-393.5)-(-1216.3) = +269.3\ \mathrm{kJ/mol}$$
>
> $$\Delta S^{\circ} = 70.42 + 213.6 - 112.1 = +171.9\ \mathrm{J/(mol\cdot K)}$$
>
> $\Delta H^{\circ}>0$、$\Delta S^{\circ}>0$ → 表 19.3 **类型 (iv)**：**升温有利于分解**（与 `习题 19.8` 同类）。因此 1100 K 的 $P_{\mathrm{CO_2}}$ 必须**远大于** 298 K 的，你的数量级方向（$10^{-39}\to10^{-11}$）对，但 (b) 的绝对值仍偏小。
>
> **正确做法**（题设「$\Delta H$、$\Delta S$ 不随温度变化」时）：
>
> $$\Delta G^{\circ}(T) \approx \Delta H^{\circ} - T\Delta S^{\circ}$$
>
> 1100 K：
>
> $$\Delta G^{\circ} \approx 269.3 - 1100\times0.1719 \approx +80.2\ \mathrm{kJ/mol}$$
>
> 比 298 K 的 $+218\ \mathrm{kJ/mol}$ **已明显下降**（高温下 $T\Delta S$ 项拉低 $\Delta G^{\circ}$），但仍为正，故 1100 K 下分解仍非自发，只是比室温「没那么不利」。
>
> $$K_p = P_{\mathrm{CO_2}} = \exp\!\bigl(-80200/(8.314\times1100)\bigr) \approx \mathbf{1.6\times10^{-4}\ \mathrm{atm}}$$
>
> 对比你的 (b) 结果 $4.4\times10^{-11}\ \mathrm{atm}$：错在仍用了 $\Delta G^{\circ}=+218\ \mathrm{kJ/mol}$，相当于**低估了温度对自发性的促进**。
>
> ### 4. 一张表串起整题
>
> | | 298 K | 1100 K |
> |---|--------|--------|
> | $\Delta G^{\circ}$ | $+218\ \mathrm{kJ/mol}$（用 $\Delta G_f^{\circ}$） | $+80\ \mathrm{kJ/mol}$（用 $\Delta H^{\circ}-T\Delta S^{\circ}$） |
> | $P_{\mathrm{CO_2}}$ | $\sim10^{-39}\ \mathrm{atm}$ | $\sim10^{-4}\ \mathrm{atm}$ |
> | 分解倾向 | 极不利 | 仍不利，但明显改善 |
>
> 平衡温度（$\Delta G^{\circ}=0$）：$T_{\mathrm{eq}} \approx \Delta H^{\circ}/\Delta S^{\circ} \approx 269300/172 \approx 1570\ \mathrm{K}$（约 $1300\ ^{\circ}\mathrm{C}$）。1100 K 仍低于此温度，故 $\Delta G^{\circ}>0$、$K<1$ 符合预期；工业上碳酸钡高温才明显分解，与估算一致。
>
> ### 5. 代码小建议
>
> - **(a)**：保持现有 $\Delta G_f^{\circ}$ 路线即可。
> - **(b)**：先由附录 C 算 $\Delta H^{\circ}$、$\Delta S^{\circ}$，再 `G = H - T*S`，最后 `K = np.exp(-G.to(ureg.J).magnitude / (R*T))`。不要复用 (a) 的 `G`。
> - 输出可并列 $\Delta G^{\circ}$ 与 $K$，便于核对。
>
> ### 6. 小结（帮你「摸准」）
>
> 1. **固体不计、$P_{\mathrm{CO_2}}=K_p$**——你已掌握。  
> 2. **298 K 题**：附录 C 的 $\Delta G_f^{\circ}$ → $\Delta G^{\circ}$ → $K$。  
> 3. **非 298 K 题**：若只给 298 K 数据，用 $\Delta G^{\circ}(T)\approx\Delta H^{\circ}-T\Delta S^{\circ}$，**不能**只改 $T$ 而保留 298 K 的 $\Delta G^{\circ}$。  
> 4. 本反应 $\Delta H^{\circ}>0$、$\Delta S^{\circ}>0$，**温度越高，$P_{\mathrm{CO_2}}$ 越大**——这是判断 (b) 算错没算错的最快自检。
>
:::


![](./assets/images/others/20260707_143848_2.webp)